<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Morteza Jafari" />

<meta name="date" content="2023-11-07" />

<title>Microbiome Project Analysis Report</title>

<script src="Microbiome-Project_files/header-attrs-2.25/header-attrs.js"></script>
<script src="Microbiome-Project_files/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="Microbiome-Project_files/bootstrap-3.3.5/css/cerulean.min.css" rel="stylesheet" />
<script src="Microbiome-Project_files/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="Microbiome-Project_files/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="Microbiome-Project_files/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="Microbiome-Project_files/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="Microbiome-Project_files/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="Microbiome-Project_files/tocify-1.9.1/jquery.tocify.js"></script>
<script src="Microbiome-Project_files/navigation-1.1/tabsets.js"></script>
<link href="Microbiome-Project_files/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="Microbiome-Project_files/highlightjs-9.12.0/highlight.js"></script>
<script src="Microbiome-Project_files/kePrint-0.0.1/kePrint.js"></script>
<link href="Microbiome-Project_files/lightable-0.0.1/lightable.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>



<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div id="header">



<h1 class="title toc-ignore">Microbiome Project Analysis Report</h1>
<h3 class="subtitle">Part one</h3>
<h4 class="author">Morteza Jafari</h4>
<h4 class="date">2023-11-07</h4>

</div>


<style type="text/css">
/* Set the background to a soft pastel and adjust text color for readability */
body {
  background-color: #f4f8fb; /* A soft pastel-like background color */
  color: #3a3a3a; /* A darker text color for better readability */
}

/* Main content font size */
p {
  font-size: 18px; /* Adjust the size as needed */
}

/* Define the font family for all headers and use the original Cerulean color */
h1, h2, h3, h4, h5, h6 {
  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
}

/* Style main section titles (h2) */
h1 {
  font-size: 30px; /* A larger font size for main titles */
  font-weight: bold; /* Make the titles thicker */
  margin-top: 20px;
  margin-bottom: 10px;
  padding-bottom: 2px;
}

/* Style main section titles (h2) */
h2 {
  font-size: 30px; /* A larger font size for main titles */
  font-weight: bold; /* Make the titles thicker */
  margin-top: 20px;
  margin-bottom: 10px;
  padding-bottom: 2px;
}

/* Additional styles can go here */

</style>
<style type="text/css">
/* Style the floating ToC */
nav[data-toggle="toc"] {
  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
  background-color: #f9f9f9; /* Light background for the ToC */
  border: 1px solid #eee; /* Add a border to the ToC */
  border-radius: 4px; /* Optionally round the corners */
  width: 250px; /* Set the width of the ToC */
}

/* Style the ToC list items */
nav[data-toggle="toc"] li {
  font-size: 14px; /* Set the font size for ToC items */
  margin-top: 2px; /* Add a bit of margin to the top of ToC items */
}

/* Style the ToC links */
nav[data-toggle="toc"] a {
  color: #337ab7; /* Color of ToC links to match Cerulean theme */
  text-decoration: none; /* Remove text underline */
}

/* Style the ToC links on hover */
nav[data-toggle="toc"] a:hover {
  color: #23527c; /* Darker color on hover */
  text-decoration: underline; /* Add underline on hover */
}

/* Current section highlight */
nav[data-toggle="toc"] .active {
  font-weight: bold; /* Make the active section bold */
  background-color: #e7f3fe; /* Light background color for the active section */
}

</style>
<div id="introduction" class="section level1">
<h1>Introduction</h1>
<p>This document outlines the process and findings of a microbiome data
analysis project aimed at practicing and familiarizing ourselves with
standard bioinformatics methods. The analysis was conducted using
resources provided by the National Supercomputer Centre (NSC) in Sweden,
accessed through a ThinLinc client terminal. The project involved
setting up the necessary computational environment, processing raw
sequence data, and performing an analysis that includes quality control,
error correction, chimera detection, and taxonomic classification. The
workflow and results presented here serve as a record of the analytical
techniques learned and applied in the study of microbial
communities.</p>
</div>
<div id="setup" class="section level1">
<h1>1. Setup</h1>
<p>This section describes the initial setup process required for the
microbiome data analysis project. It includes loading necessary modules,
booking computational resources, copying environment files, creating a
conda environment, and setting up the directory structure for the
project.</p>
<div id="load-required-modules" class="section level2">
<h2>Load Required Modules</h2>
<p>First, we load the necessary Anaconda module which provides the conda
package and environment management system.</p>
<pre class="bash"><code>module load Anaconda/2022.05-nsc1</code></pre>
</div>
<div id="book-computational-resources" class="section level2">
<h2>Book Computational Resources</h2>
<p>Next, we request an interactive node with the specified resources for
a duration of 4 hours. This step is performed on the command line and is
specific to systems using a job scheduler like Slurm.</p>
<pre class="bash"><code>interactive -t 04:00:00 -n2 --mem=16G --cpus-per-task=4 --reservation=devel</code></pre>
</div>
<div id="copy-environment-file" class="section level2">
<h2>Copy Environment File</h2>
<p>We then copy the environment file from a shared directory to our
personal workspace.</p>
<pre class="bash"><code>cp /proj/liu-compute-2023-21/16S_amplicon/environment/environment.yaml /proj/liu-compute-2023-21/users/x_morja/</code></pre>
</div>
<div id="create-and-activate-conda-environment" class="section level2">
<h2>Create and Activate Conda Environment</h2>
<p>Creating and activating a new conda environment using the copied
environment file ensures that we have all the necessary
dependencies.</p>
<pre class="bash"><code>conda env create -f environment.yaml
conda activate 16S-Part1</code></pre>
</div>
<div id="directory-setup" class="section level2">
<h2>Directory Setup</h2>
<p>Finally, we navigate to the user directory and create a new project
directory with a structured subdirectory layout for the analysis.</p>
<pre class="bash"><code>cd /proj/liu-compute-2023-21/users/x_morja/
mkdir Microbiome_Project
cd Microbiome_Project
mkdir -p 16S_Part1/quality_check_raw 16S_Part1/trimmed_data 16S_Part1/quality_check_trimmed 16S_Part1/Routput</code></pre>
</div>
</div>
<div id="quality-check" class="section level1">
<h1>2. Quality Check</h1>
<p>Quality control is a critical first step in any bioinformatics
analysis. It allows us to assess the quality of the raw sequencing data
and identify any potential issues that may affect the subsequent
analysis. The following steps were performed to ensure the quality of
our 16S rRNA gene sequencing data.</p>
<div id="fastqc-analysis" class="section level2">
<h2>FastQC Analysis</h2>
<p>FastQC is a widely used tool for quality checking of raw sequence
data. It provides a simple way to assess the quality of your data before
moving on to the analysis. It generates various statistics and
visualizations that help in identifying problems like poor quality
scores, overrepresented sequences, and potential contamination.</p>
<pre class="bash"><code># Run FastQC for quality check on raw data
fastqc -o /proj/liu-compute-2023-21/users/x_morja/Microbiome_Project/16S_Part1/quality_check_raw /proj/liu-compute-2023-21/16S_amplicon/raw_data/data1/*.fastq.gz</code></pre>
</div>
<div id="multiqc-summary" class="section level2">
<h2>MultiQC Summary</h2>
<p>After running FastQC on multiple samples, it can be tedious to go
through each report individually. MultiQC aggregates results from
bioinformatics analyses across many samples into a single report. It is
a convenient tool for summarizing the output from FastQC, making it
easier to assess and compare the quality of multiple samples at
once.</p>
<pre class="bash"><code># Run MultiQC to compile the FastQC reports into a single report
multiqc /proj/liu-compute-2023-21/users/x_morja/Microbiome_Project/16S_Part1/quality_check_raw -o /proj/liu-compute-2023-21/users/x_morja/Microbiome_Project/16S_Part1/quality_check_raw</code></pre>
</div>
<div id="viewing-the-multiqc-report" class="section level2">
<h2>Viewing the MultiQC Report</h2>
<p>The MultiQC report is generated as an HTML file, which can be opened
with any web browser. This step involves navigating to the directory
containing the report and opening it for review.</p>
<pre class="bash"><code># Navigate to the directory containing the MultiQC report
cd /proj/liu-compute-2023-21/users/x_morja/Microbiome_Project/16S_Part1/quality_check_raw

# List HTML files to confirm the presence of the MultiQC report
ls -l *.html

# Open the MultiQC report using the default web browser
xdg-open multiqc_report.html</code></pre>
</div>
<div id="transferring-the-report-to-a-local-machine"
class="section level2">
<h2>Transferring the Report to a Local Machine</h2>
<p>For further analysis or record-keeping, it may be necessary to
transfer files from the remote server to a local machine. The
<code>scp</code> command securely copies files between hosts over the
network.</p>
<pre class="bash"><code># Securely copy the MultiQC report to the local machine
scp x_morja@sigma.nsc.liu.se:/proj/liu-compute-2023-21/users/x_morja/Microbiome_Project/16S_Part1/quality_check_raw/multiqc_report.html D:\LiU\Masters\Bioinformatics\Microbiome</code></pre>
</div>
</div>
<div id="multiqc-result-interpretation-for-raw-data"
class="section level1">
<h1>3. MultiQC Result Interpretation for Raw Data</h1>
<p>This section delves into the key metrics from the MultiQC report,
which consolidates the quality control data assessed by FastQC. The
report’s metrics are vital for validating the sequencing data’s
integrity, which, in turn, supports the accuracy of any subsequent
analysis.</p>
<div id="raw-multiqc-report-embedding" class="section level2">
<h2>Raw MultiQC Report Embedding</h2>
<p>This section delineates the initial results from the MultiQC report,
which encapsulates the baseline quality control metrics for our
sequencing data before any filtering and trimming procedures have been
applied. The report provides an essential assessment of the raw
sequencing output, offering insights into the intrinsic quality of the
data, such as the initial sequence quality scores, GC content, sequence
length, and the presence of adapter sequences. These preliminary metrics
are critical for gauging the native state of our sequencing reads and
setting a benchmark for the necessary quality improvement steps.
Understanding the starting quality of our data is pivotal for subsequent
optimization through filtering and trimming, ensuring that downstream
analyses in our microbiome research are built upon a foundation of
robust and unadulterated data.</p>
<iframe src="https://rawcdn.githack.com/TakaLiU/Microbiome/3e10bfa8a9c7b67df63942a80394cbc1642e733c/multiqc_report.html" height="600px" width="100%" frameborder="0"></iframe>
</div>
<div id="general-statistics" class="section level2">
<h2>General Statistics</h2>
<p><strong>Table 1:</strong> This table presents the aggregated metrics
of pre-quality control sequencing data as generated by MultiQC analysis.
The table enumerates duplication rates, GC content, read length
statistics, percentage of failed quality checks, and sequencing depth
across various samples (only two exemplified here), with additional data
elided for brevity. The duplication rates are alarmingly high across all
samples, indicating potential amplification bias or low library
diversity, which is a critical consideration for further processing
steps. GC content is notably stable across the dataset, suggesting a
balanced representation of nucleotide composition. Read lengths are
technically consistent, signifying the standardization of sequencing
procedures. Despite these consistencies, a significant failure rate in
quality checks for some samples, like Mock1_S96_L001_R1_001, underscores
the presence of low-quality bases that may affect subsequent analyses.
Sequencing depth variability is evident, with some samples showing
depths as low as zero, calling into question the uniformity of sequence
capture and necessitating further investigative or corrective actions.
Collectively, the data underscore the imperative for meticulous scrutiny
and the application of stringent quality control measures before
advancing to downstream analytical applications.</p>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; overflow-x: scroll; width:100%; ">
<table class="table table-striped table-hover table-condensed table-responsive" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
Sample.Name
</th>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
X..Dups
</th>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
X..GC
</th>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
Average.Read.Length
</th>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
Median.Read.Length
</th>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
X..Failed
</th>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
M.Seqs
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
mock1_S96_L001_R1_001
</td>
<td style="text-align:center;">
93.8%
</td>
<td style="text-align:center;">
52%
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
55%
</td>
<td style="text-align:center;">
0.0
</td>
</tr>
<tr>
<td style="text-align:center;">
mock1_S96_L001_R2_001
</td>
<td style="text-align:center;">
93.7%
</td>
<td style="text-align:center;">
53%
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
36%
</td>
<td style="text-align:center;">
0.0
</td>
</tr>
<tr>
<td style="text-align:center;">
mock2_S96_L001_R1_001
</td>
<td style="text-align:center;">
95.6%
</td>
<td style="text-align:center;">
52%
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
36%
</td>
<td style="text-align:center;">
0.2
</td>
</tr>
<tr>
<td style="text-align:center;">
mock2_S96_L001_R2_001
</td>
<td style="text-align:center;">
94.4%
</td>
<td style="text-align:center;">
53%
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
36%
</td>
<td style="text-align:center;">
0.2
</td>
</tr>
<tr>
<td style="text-align:center;">
mock3_S69_L001_R1_001
</td>
<td style="text-align:center;">
96.3%
</td>
<td style="text-align:center;">
52%
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
36%
</td>
<td style="text-align:center;">
0.2
</td>
</tr>
<tr>
<td style="text-align:center;">
mock3_S69_L001_R2_001
</td>
<td style="text-align:center;">
94.8%
</td>
<td style="text-align:center;">
53%
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
36%
</td>
<td style="text-align:center;">
0.2
</td>
</tr>
<tr>
<td style="text-align:center;">
mock4_S96_L001_R1_001
</td>
<td style="text-align:center;">
95.3%
</td>
<td style="text-align:center;">
53%
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
36%
</td>
<td style="text-align:center;">
0.2
</td>
</tr>
<tr>
<td style="text-align:center;">
mock4_S96_L001_R2_001
</td>
<td style="text-align:center;">
94.2%
</td>
<td style="text-align:center;">
53%
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
36%
</td>
<td style="text-align:center;">
0.2
</td>
</tr>
<tr>
<td style="text-align:center;">
mock5_S54_L001_R1_001
</td>
<td style="text-align:center;">
92.8%
</td>
<td style="text-align:center;">
53%
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
36%
</td>
<td style="text-align:center;">
0.1
</td>
</tr>
<tr>
<td style="text-align:center;">
mock5_S54_L001_R2_001
</td>
<td style="text-align:center;">
89.2%
</td>
<td style="text-align:center;">
53%
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
45%
</td>
<td style="text-align:center;">
0.1
</td>
</tr>
<tr>
<td style="text-align:center;">
mock6_S46_L001_R1_001
</td>
<td style="text-align:center;">
95.7%
</td>
<td style="text-align:center;">
53%
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
36%
</td>
<td style="text-align:center;">
0.2
</td>
</tr>
<tr>
<td style="text-align:center;">
mock6_S46_L001_R2_001
</td>
<td style="text-align:center;">
93.9%
</td>
<td style="text-align:center;">
53%
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
36%
</td>
<td style="text-align:center;">
0.2
</td>
</tr>
<tr>
<td style="text-align:center;">
mock7_S46_L001_R1_001
</td>
<td style="text-align:center;">
96.4%
</td>
<td style="text-align:center;">
53%
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
36%
</td>
<td style="text-align:center;">
0.4
</td>
</tr>
<tr>
<td style="text-align:center;">
mock7_S46_L001_R2_001
</td>
<td style="text-align:center;">
95.0%
</td>
<td style="text-align:center;">
53%
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
36%
</td>
<td style="text-align:center;">
0.4
</td>
</tr>
<tr>
<td style="text-align:center;">
mock8_S33_L001_R1_001
</td>
<td style="text-align:center;">
94.8%
</td>
<td style="text-align:center;">
52%
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
45%
</td>
<td style="text-align:center;">
0.1
</td>
</tr>
<tr>
<td style="text-align:center;">
mock8_S33_L001_R2_001
</td>
<td style="text-align:center;">
93.4%
</td>
<td style="text-align:center;">
53%
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
36%
</td>
<td style="text-align:center;">
0.1
</td>
</tr>
<tr>
<td style="text-align:center;">
mock9_S48_L001_R1_001
</td>
<td style="text-align:center;">
94.7%
</td>
<td style="text-align:center;">
53%
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
45%
</td>
<td style="text-align:center;">
0.2
</td>
</tr>
<tr>
<td style="text-align:center;">
mock9_S48_L001_R2_001
</td>
<td style="text-align:center;">
92.9%
</td>
<td style="text-align:center;">
53%
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
301 bp
</td>
<td style="text-align:center;">
36%
</td>
<td style="text-align:center;">
0.2
</td>
</tr>
</tbody>
</table>
</div>
<p>Based on the general statistics table above, there are several points
of concern that merit attention:</p>
<p>The summary statistics from the MultiQC report provide a quantitative
overview of the sequencing data prior to quality filtering and trimming.
The data indicate a pervasive presence of duplicate sequences, with
duplication rates surpassing 90% for all samples, as exemplified by the
mock1_S96_L001_R1_001 sample at 93.8%. These rates may suggest an
amplification bias or a limited diversity within the sample library. A
consistent GC content around 52-53% across samples, including
mock2_S96_L001_R1_001 at 52%, implies an unbiased representation of
guanine-cytosine nucleotides, which is crucial for avoiding sequencing
artifacts.</p>
<p>The uniformity in sequence length, with both average and median read
lengths at 301 base pairs for all datasets, reflects the technical
consistency of the sequencing process. However, the percentage of failed
quality checks, particularly notable in mock1_S96_L001_R1_001 with a 55%
failure rate, indicates a considerable fraction of low-quality base
calls, which could potentially impact the reliability of downstream
analyses.</p>
<p>Sequencing depth, denoted by the ‘M Seqs’ metric, reveals a range
from 0.0 to 0.4 million sequences, with certain samples like
mock7_S46_L001_R1_001 displaying a higher sequencing depth of 0.4
million sequences. This variation in sequencing depth, particularly the
instances of zero sequences reported, raises concerns about the
consistency of the sequencing output and may necessitate additional
examination or a repetition of the sequencing run to confirm data
quality.</p>
<p>In conclusion, the initial assessment of the sequencing data reveals
high duplication rates and a notable proportion of low-quality reads
that need to be addressed. While GC content and read length distribution
appear satisfactory, the failure rates and inconsistencies in sequencing
depth highlight the need for a careful review of the sequencing
protocols and potentially the implementation of additional quality
control measures. The subsequent quality filtering and trimming steps
are expected to enhance the data quality by eliminating low-quality
sequences and reducing duplication rates, thus refining the dataset for
accurate biological interpretation.</p>
</div>
<div id="sequence-quality-histograms" class="section level2">
<h2>Sequence Quality Histograms</h2>
<p><img src="Output/RawQC/Figures/fastqc_per_base_sequence_quality_plot.jpeg" width="100%" />
<strong>Figure 1:</strong> Per Base Sequence Quality Distribution Across
Sequencing Reads. The plot represents the distribution of mean Phred
quality scores at each base position across all sequencing reads. Each
line corresponds to one of the sequenced samples, indicating the average
base call accuracy as a function of the position in the read. The
initial positions show high-quality scores, predominantly above a Phred
score of 30, indicating a base call accuracy of 99.9%. The quality
remains consistent and well above the threshold Phred score of 20, which
corresponds to a 99% base call accuracy, for the majority of the read
length. Towards the 3’ end of the reads, there is a characteristic
decline in quality scores, which is a common observation in
high-throughput sequencing datasets. However, the decline in quality
scores remains above the Phred score of 20 until approximately the 250th
base pair, after which a more pronounced drop is observed. The shaded
regions demarcate the quality score ranges, with green representing
scores above 30, yellow for scores between 20 and 30, and red indicating
scores below 20. The consistent high-quality scores across the initial
and mid-regions of the reads are indicative of the overall excellent
sequencing quality achieved, while the late decline suggests the
necessity for cautious interpretation of base calls at the end of the
reads or potential read trimming for downstream applications.</p>
<p>In Figure 1, the Mean Quality Scores plot derived from FastQC
analysis is presented, showcasing the Phred quality scores across the
length of sequencing reads. The data commences with scores well above
30, indicative of a high level of confidence in base identification at
the initial stages of the reads. This high-quality scoring is sustained
throughout a significant portion of the sequencing run, maintaining well
above the threshold of 20, which is generally accepted as the standard
for high-quality base calls. Such consistency ensures the reliability of
the majority of the read length and is suggestive of robust sequencing
chemistry and effective base calling algorithms employed during the
sequencing process.</p>
<p>Towards the terminal end of the reads, a decline in quality scores is
observed, which is a typical manifestation in sequencing data due to the
diminishing fidelity of sequencing reactions over time. Notably, this
decline remains above the quality threshold for the majority of the
read, with scores only dipping below the Phred score of 20 past the 250
base pair mark. This pattern is consistent across the dataset,
manifesting in all samples, which is an encouraging sign that the
end-read quality decline does not overshadow the overall data
integrity.</p>
<p>The implications of the observed quality score distribution are
two-fold; first, the data preceding the decline represents high-fidelity
sequencing output suitable for accurate downstream applications such as
variant analysis or transcript assembly. Second, the latter part of the
reads may require cautious consideration or trimming to ensure the
highest data quality in subsequent analyses. The observed quality scores
support the notion that, with appropriate quality control measures, the
sequencing data can be deemed highly reliable for rigorous biological
examination and interpretation.</p>
</div>
<div id="status-checks" class="section level2">
<h2>Status Checks</h2>
<p><img src="Output/RawQC/Figures/fastqc-status-check-heatmap.jpeg" width="100%" />
<strong>Figure 2:</strong> FastQC Status Checks Heatmap for Sequencing
Data Quality. The heatmap summarizes the outcomes of quality control
checks for nine mock sequencing samples (mock1_S96_L001_R1_00 to
mock9_S48_L001_R1_00). Each row represents a distinct quality metric,
while each column corresponds to a sample. The color-coded system
indicates the status of each metric: green (pass), yellow (warn), and
red (fail). Metrics evaluated include base sequence quality, GC content,
sequence length distribution, and adapter content. The consistent green
across ‘Per Base Sequence Content’ and ‘Per Sequence GC Content’ metrics
suggests acceptable nucleotide distribution and GC composition.
Occurrences of yellow and red in ‘Per Base N Content’ and ‘Sequence
Length Distribution’ point to potential concerns regarding base call
accuracy and read length uniformity. The ‘Adapter Content’ metric
predominantly displays green, indicating effective adapter sequence
removal. This comprehensive quality overview aids in the identification
of samples and metrics requiring further scrutiny to ensure the
integrity of downstream data analysis.</p>
<p>In Figure 2, the FastQC Status Checks heatmap provides a
comprehensive visualization of the quality metric outcomes for multiple
samples in a sequencing batch. The heatmap is color-coded: green
represents a pass, yellow indicates a warning, and red denotes a fail.
The overview of the quality control checks is crucial for assessing the
immediate integrity of the sequencing data. For instance, the ‘Per Base
Sequence Content’ and ‘Per Sequence GC Content’ checks exhibit a
preponderance of green across all samples, suggesting that the
nucleotide composition per base is consistent and the GC content is
within expected ranges, reflecting a lack of bias that could compromise
the sequencing analysis.</p>
<p>Conversely, several samples such as ‘mock2_S96_L001_R1_00’ and
‘mock3_S69_L001_R1_00’ show a concerning number of warnings and failures
in sections such as ‘Per Base N Content’ and ‘Sequence Length
Distribution’. These warnings may indicate low-quality base calls at
certain positions and unexpected variation in read lengths,
respectively. Such findings warrant a closer examination to ensure that
any analytical conclusions drawn from this data are not marred by
underlying quality issues.</p>
<p>The heatmap also draws attention to the ‘Adapter Content’ quality
check, where a majority of the samples have passed, implying the
successful removal of sequence adapters during preprocessing. This
suggests that subsequent data analyses will not be confounded by
artificial sequences, allowing for a more accurate representation of the
biological samples. Overall, the heatmap serves as an efficient tool for
the preliminary assessment of data quality, highlighting areas that may
require further attention before proceeding with more advanced
bioinformatics analyses.</p>
</div>
</div>
<div id="quality-filtering-and-trimming" class="section level1">
<h1>4. Quality Filtering and Trimming</h1>
<p>This section outlines the steps taken to filter and trim the
sequencing data to ensure high-quality reads for downstream analysis.
The process involves creating a list of sample names, trimming primers
and low-quality ends, and then further trimming to a higher quality
threshold.</p>
<pre class="r"><code># Navigate to the directory containing raw data and create a file with sample names
cd /proj/liu-compute-2023-21/16S_amplicon/raw_data/data1
ls *fastq.gz | cut -f1-2 -d &quot;_&quot; &gt; /proj/liu-compute-2023-21/users/x_morja/Microbiome_Project/16S_Part1/samples

# Return to the working directory for the project
cd /proj/liu-compute-2023-21/users/x_morja/Microbiome_Project/16S_Part1

# The first round of trimming removes primers and trims the 5&#39; end to a quality score of 35
# The &#39;bbduk.sh&#39; script is part of the BBTools suite and is used for read trimming
# &#39;literal&#39; specifies the sequences to trim, &#39;k&#39; is the k-mer length for matching, &#39;ordered&#39; keeps reads in the same order,
# &#39;ktrim&#39; specifies trimming at the left end, &#39;copyundefined&#39; includes undefined bases in k-mers, &#39;ftm&#39; trims to multiple of 5,
# &#39;qtrim&#39; specifies quality trimming at the left end, &#39;trimq&#39; is the quality threshold, and &#39;minlen&#39; is the minimum read length
for sample in $(cat samples); do 
    echo &quot;Trimming sample: $sample&quot;
    bbduk.sh in=/proj/liu-compute-2023-21/16S_amplicon/raw_data/data1/&quot;$sample&quot;_L001_R1_001.fastq.gz \
             in2=/proj/liu-compute-2023-21/16S_amplicon/raw_data/data1/&quot;$sample&quot;_L001_R2_001.fastq.gz \
             out=trimmed_data/&quot;$sample&quot;_R1_trimmed1.fastq.gz \
             out2=trimmed_data/&quot;$sample&quot;_R2_trimmed1.fastq.gz \
             literal=CCTACGGGNGGCWGCAG,GACTACHVGGGTATCTAATCC k=17 ordered=t ktrim=l \
             copyundefined ftm=5 qtrim=l trimq=35 minlen=100
done 2&gt; trimming1_stats.txt

# The second round of trimming further processes the reads trimmed in the first round
# This step trims the 3&#39; end to a quality score of 30 to ensure all bases are of high quality
# &#39;qtrim=r&#39; specifies quality trimming at the right end
for sample in $(cat samples); do 
    echo &quot;Further trimming sample: $sample&quot;
    bbduk.sh in=trimmed_data/&quot;$sample&quot;_R1_trimmed1.fastq.gz \
             in2=trimmed_data/&quot;$sample&quot;_R2_trimmed1.fastq.gz \
             out=trimmed_data/&quot;$sample&quot;_R1_trimmed2.fastq.gz \
             out2=trimmed_data/&quot;$sample&quot;_R2_trimmed2.fastq.gz \
             qtrim=r trimq=30 minlen=100
done 2&gt; trimming2_stats.txt</code></pre>
</div>
<div id="check-the-quality-of-trimmed-data" class="section level1">
<h1>5. Check the Quality of Trimmed Data</h1>
<p>After trimming the raw sequencing data for quality, it is essential
to assess the quality of the trimmed data. This step ensures that the
trimming process has effectively improved the quality of the reads and
that they are suitable for downstream analysis.</p>
<pre class="r"><code># Navigate to the directory containing the trimmed sequencing data
cd /proj/liu-compute-2023-21/users/x_morja/Microbiome_Project/16S_Part1/trimmed_data/

# Execute FastQC on all trimmed fastq.gz files and direct the output to a specific directory
fastqc -o /proj/liu-compute-2023-21/users/x_morja/Microbiome_Project/16S_Part1/quality_check_trimmed /proj/liu-compute-2023-21/users/x_morja/Microbiome_Project/16S_Part1/trimmed_data/*trimmed2.fastq.gz

# Use MultiQC to aggregate individual FastQC reports into a comprehensive single report
multiqc /proj/liu-compute-2023-21/users/x_morja/Microbiome_Project/16S_Part1/quality_check_trimmed -o /proj/liu-compute-2023-21/users/x_morja/Microbiome_Project/16S_Part1/quality_check_trimmed

# If the quality of the trimmed data is satisfactory, the intermediate files can be removed to save space
rm /proj/liu-compute-2023-21/users/x_morja/Microbiome_Project/16S_Part1/trimmed_data/*trimmed1.fastq.gz

# Securely copy the final quality report to a local machine for further analysis and record-keeping
scp x_morja@sigma.nsc.liu.se:/proj/liu-compute-2023-21/users/x_morja/Microbiome_Project/16S_Part1/quality_check_trimmed/multiqc_report.html D:\LiU\Masters\Bioinformatics\Microbiome</code></pre>
</div>
<div id="comparative-multiqc-result-interpretation-raw-vs.-trimmed"
class="section level1">
<h1>6. Comparative MultiQC Result Interpretation: Raw vs. Trimmed</h1>
<p>This section provides a comparative analysis of the MultiQC reports
generated before and after the quality filtering and trimming processes.
The quality filtering and trimming steps are designed to enhance data
integrity by removing low-quality bases, adapter sequences, and other
potential sources of error. The expectation is that these preprocessing
steps will result in improved quality scores, reduced error rates, and
the elimination of artifacts that could compromise downstream analyses.
A thorough comparison of the MultiQC metrics pre- and post-trimming
allows us to assess the impact of these procedures on data quality. Key
indicators include the proportion of high-quality reads, the consistency
of read lengths, and the reduction of duplicate sequences. The outcome
of this comparison will guide the decision-making process for subsequent
analytical steps and provide assurance that the data is processed to
meet the stringent requirements for accurate microbial community
analysis.</p>
<div id="trimmed-multiqc-report-embedding" class="section level2">
<h2>Trimmed MultiQC Report Embedding</h2>
<p>This section presents the results from the MultiQC report, a
comprehensive tool that consolidates quality control metrics from
various bioinformatics analyses into a unified report. Here, we focus on
the post-QC stages of filtered and trimmed sequences, a critical
juncture in high-throughput sequencing workflows where sequences are
refined to eliminate low-quality bases and adapter sequences. The
MultiQC outcomes provide an overview of the quality scores, GC content,
sequence length, and adapter content post-filtering, offering a snapshot
of the data’s integrity. These results are instrumental for validating
the effectiveness of the QC process and ensuring the preparedness of the
dataset for advanced downstream applications such as taxonomic profiling
and variant analysis, which demand high-quality input for accurate and
reliable interpretation in microbiome research.</p>
<iframe src="https://rawcdn.githack.com/TakaLiU/Microbiome/3e10bfa8a9c7b67df63942a80394cbc1642e733c/Trimmed_multiqc_report.html" height="600px" width="100%" frameborder="0"></iframe>
</div>
<div id="general-statistics-1" class="section level2">
<h2>General Statistics</h2>
<p><strong>Table 2:</strong> Table Description: Table 2 displays the
summary statistics post-quality filtering and trimming, highlighting
changes in duplication rates, average and median read lengths, and
percentage of sequences failing quality checks. The data show a
reduction in read lengths and unchanged high duplication rates,
indicating effective trimming of low-quality bases and unchanged
sequence complexity. The consistent sequencing depth after trimming
confirms the preservation of data quantity for reliable downstream
analysis.</p>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; overflow-x: scroll; width:100%; ">
<table class="table table-striped table-hover table-condensed table-responsive" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
Sample.Name
</th>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
X..Dups
</th>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
X..GC
</th>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
Average.Read.Length
</th>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
Median.Read.Length
</th>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
X..Failed
</th>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
M.Seqs
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
mock1_S96_R1_trimmed2
</td>
<td style="text-align:center;">
94.4%
</td>
<td style="text-align:center;">
51%
</td>
<td style="text-align:center;">
217 bp
</td>
<td style="text-align:center;">
227 bp
</td>
<td style="text-align:center;">
45%
</td>
<td style="text-align:center;">
0.0
</td>
</tr>
<tr>
<td style="text-align:center;">
mock1_S96_R2_trimmed2
</td>
<td style="text-align:center;">
95.6%
</td>
<td style="text-align:center;">
52%
</td>
<td style="text-align:center;">
162 bp
</td>
<td style="text-align:center;">
162 bp
</td>
<td style="text-align:center;">
45%
</td>
<td style="text-align:center;">
0.0
</td>
</tr>
<tr>
<td style="text-align:center;">
mock2_S96_R1_trimmed2
</td>
<td style="text-align:center;">
96.8%
</td>
<td style="text-align:center;">
51%
</td>
<td style="text-align:center;">
242 bp
</td>
<td style="text-align:center;">
262 bp
</td>
<td style="text-align:center;">
36%
</td>
<td style="text-align:center;">
0.1
</td>
</tr>
<tr>
<td style="text-align:center;">
mock2_S96_R2_trimmed2
</td>
<td style="text-align:center;">
97.2%
</td>
<td style="text-align:center;">
52%
</td>
<td style="text-align:center;">
179 bp
</td>
<td style="text-align:center;">
182 bp
</td>
<td style="text-align:center;">
45%
</td>
<td style="text-align:center;">
0.1
</td>
</tr>
<tr>
<td style="text-align:center;">
mock3_S69_R1_trimmed2
</td>
<td style="text-align:center;">
97.1%
</td>
<td style="text-align:center;">
51%
</td>
<td style="text-align:center;">
245 bp
</td>
<td style="text-align:center;">
277 bp
</td>
<td style="text-align:center;">
36%
</td>
<td style="text-align:center;">
0.2
</td>
</tr>
<tr>
<td style="text-align:center;">
mock3_S69_R2_trimmed2
</td>
<td style="text-align:center;">
97.5%
</td>
<td style="text-align:center;">
52%
</td>
<td style="text-align:center;">
188 bp
</td>
<td style="text-align:center;">
197 bp
</td>
<td style="text-align:center;">
45%
</td>
<td style="text-align:center;">
0.2
</td>
</tr>
<tr>
<td style="text-align:center;">
mock4_S96_R1_trimmed2
</td>
<td style="text-align:center;">
96.6%
</td>
<td style="text-align:center;">
51%
</td>
<td style="text-align:center;">
238 bp
</td>
<td style="text-align:center;">
257 bp
</td>
<td style="text-align:center;">
36%
</td>
<td style="text-align:center;">
0.1
</td>
</tr>
<tr>
<td style="text-align:center;">
mock4_S96_R2_trimmed2
</td>
<td style="text-align:center;">
96.8%
</td>
<td style="text-align:center;">
52%
</td>
<td style="text-align:center;">
195 bp
</td>
<td style="text-align:center;">
202 bp
</td>
<td style="text-align:center;">
45%
</td>
<td style="text-align:center;">
0.1
</td>
</tr>
<tr>
<td style="text-align:center;">
mock5_S54_R1_trimmed2
</td>
<td style="text-align:center;">
94.6%
</td>
<td style="text-align:center;">
51%
</td>
<td style="text-align:center;">
244 bp
</td>
<td style="text-align:center;">
267 bp
</td>
<td style="text-align:center;">
36%
</td>
<td style="text-align:center;">
0.0
</td>
</tr>
<tr>
<td style="text-align:center;">
mock5_S54_R2_trimmed2
</td>
<td style="text-align:center;">
94.0%
</td>
<td style="text-align:center;">
52%
</td>
<td style="text-align:center;">
182 bp
</td>
<td style="text-align:center;">
187 bp
</td>
<td style="text-align:center;">
45%
</td>
<td style="text-align:center;">
0.0
</td>
</tr>
<tr>
<td style="text-align:center;">
mock6_S46_R1_trimmed2
</td>
<td style="text-align:center;">
96.9%
</td>
<td style="text-align:center;">
51%
</td>
<td style="text-align:center;">
228 bp
</td>
<td style="text-align:center;">
242 bp
</td>
<td style="text-align:center;">
45%
</td>
<td style="text-align:center;">
0.1
</td>
</tr>
<tr>
<td style="text-align:center;">
mock6_S46_R2_trimmed2
</td>
<td style="text-align:center;">
97.0%
</td>
<td style="text-align:center;">
52%
</td>
<td style="text-align:center;">
176 bp
</td>
<td style="text-align:center;">
182 bp
</td>
<td style="text-align:center;">
45%
</td>
<td style="text-align:center;">
0.1
</td>
</tr>
<tr>
<td style="text-align:center;">
mock7_S46_R1_trimmed2
</td>
<td style="text-align:center;">
97.5%
</td>
<td style="text-align:center;">
51%
</td>
<td style="text-align:center;">
237 bp
</td>
<td style="text-align:center;">
257 bp
</td>
<td style="text-align:center;">
45%
</td>
<td style="text-align:center;">
0.2
</td>
</tr>
<tr>
<td style="text-align:center;">
mock7_S46_R2_trimmed2
</td>
<td style="text-align:center;">
97.7%
</td>
<td style="text-align:center;">
52%
</td>
<td style="text-align:center;">
191 bp
</td>
<td style="text-align:center;">
197 bp
</td>
<td style="text-align:center;">
45%
</td>
<td style="text-align:center;">
0.2
</td>
</tr>
<tr>
<td style="text-align:center;">
mock8_S33_R1_trimmed2
</td>
<td style="text-align:center;">
96.4%
</td>
<td style="text-align:center;">
51%
</td>
<td style="text-align:center;">
228 bp
</td>
<td style="text-align:center;">
237 bp
</td>
<td style="text-align:center;">
36%
</td>
<td style="text-align:center;">
0.1
</td>
</tr>
<tr>
<td style="text-align:center;">
mock8_S33_R2_trimmed2
</td>
<td style="text-align:center;">
96.4%
</td>
<td style="text-align:center;">
52%
</td>
<td style="text-align:center;">
170 bp
</td>
<td style="text-align:center;">
172 bp
</td>
<td style="text-align:center;">
45%
</td>
<td style="text-align:center;">
0.1
</td>
</tr>
<tr>
<td style="text-align:center;">
mock9_S48_R1_trimmed2
</td>
<td style="text-align:center;">
97.0%
</td>
<td style="text-align:center;">
51%
</td>
<td style="text-align:center;">
208 bp
</td>
<td style="text-align:center;">
212 bp
</td>
<td style="text-align:center;">
45%
</td>
<td style="text-align:center;">
0.1
</td>
</tr>
<tr>
<td style="text-align:center;">
mock9_S48_R2_trimmed2
</td>
<td style="text-align:center;">
96.4%
</td>
<td style="text-align:center;">
52%
</td>
<td style="text-align:center;">
154 bp
</td>
<td style="text-align:center;">
152 bp
</td>
<td style="text-align:center;">
45%
</td>
<td style="text-align:center;">
0.1
</td>
</tr>
</tbody>
</table>
</div>
<p>Upon examination of the provided summary statistics from the MultiQC
report, both before and after quality filtering and trimming, several
observations can be made regarding the impact of these preprocessing
steps on the sequencing data.</p>
<p>Before preprocessing, the data exhibit a relatively high duplication
rate, with the percentage of duplicates (% Dups) for each sample
typically exceeding 90%, and average read lengths consistently at 301
base pairs (bp). For instance, the sample
<code>mock1_S96_L001_R1_001</code> shows a duplication rate of 93.8% and
an average read length of 301 bp. The high duplication rates are
suggestive of a limited complexity within the sequencing libraries,
which could lead to biases in representation of sequences within the
sample set. The uniform read lengths reflect the expected output from
the sequencing platform.</p>
<p>Post-trimming, the duplication rates remain high, which may be
indicative of either the nature of the samples being rich in certain
sequences or potential over-amplification during PCR. However, the
average read lengths have been reduced, as seen in
<code>mock1_S96_R1_trimmed2</code>, which now has an average read length
of 217 bp and a duplication rate of 94.4%. This reduction in read length
is expected as the trimming process aims to remove low-quality bases
predominantly found at the read ends. A notable aspect is the median
read length, which shows variation post-trimming, such as in
<code>mock2_S96_R1_trimmed2</code>, where it is 262 bp, indicating that
while the shortest reads have been trimmed, a significant proportion of
longer reads remains. The decrease in read length and the maintenance of
high duplication rates suggest that the trimming was effective in
removing low-quality bases without significantly altering the underlying
sequence composition.</p>
<p>The percentage of sequences failing quality checks (% Failed) remains
largely unchanged post-trimming, which could be due to the stringent
quality threshold (Q30) applied during the trimming process. This
threshold ensures that only high-quality sequences are retained,
although the fact that the failure rate did not decrease post-trimming
could also imply that the majority of low-quality bases were situated at
the ends of the reads. The ‘M Seqs’ metric, which is indicative of the
sequencing depth, does not show a significant change post-trimming,
suggesting that the read quantity is preserved to a large extent.</p>
<p>In summary, the trimming and quality filtering process has
effectively shortened the read lengths by removing bases of lower
quality, as expected. However, the high duplication rates persist, which
may require further investigation into library preparation methods or
the nature of the samples themselves. The consistent failure rates
post-trimming suggest that the remaining data is of high quality,
suitable for downstream analyses. The overall impact of these
preprocessing steps has refined the dataset, ensuring that subsequent
analyses are based on sequences of reliable quality while maintaining
the depth of sequencing coverage.</p>
</div>
<div id="sequence-quality-histograms-1" class="section level2">
<h2>Sequence Quality Histograms</h2>
<p><img src="Output/TrimmedQC/Figures/trimmed_fastqc_per_base_sequence_quality_plot.jpeg" width="100%" />
<strong>Figure 3:</strong> Post-Quality Filtering and Trimming Sequence
Quality Histogram. This histogram displays the distribution of mean
Phred quality scores across all base positions in the sequencing reads
following quality filtering and trimming procedures. The trimming was
performed with a threshold of Q35 at the 5’ end and Q30 at the 3’ end to
remove bases below these quality thresholds, with an additional
constraint on minimum length retention of 100 base pairs. The vertical
axis represents the Phred quality score, a logarithmic scale of base
calling error probabilities, while the horizontal axis corresponds to
the base pair positions along the read length. The lines represent
individual samples post-trimming, showcasing the quality score
maintenance throughout the sequencing reads. Notably, the plot
illustrates a preservation of high-quality scores, generally above the
Phred score of 20, across the read length for all samples, indicating
successful trimming of low-quality sequence ends and retention of
high-quality bases suitable for accurate downstream analysis. The
improved quality profile post-trimming ensures the robustness of the
sequence data for further bioinformatic workflows.</p>
<p>The pre- and post-quality filtering and trimming sequence quality
histograms provide a visual account of the sequencing data’s integrity.
The pre-filtering histogram depicts relatively high-quality scores
across the majority of the sequencing reads with a Phred score
consistently above 30, indicating a high confidence in the base calls.
However, a notable decline in quality is observed towards the end of the
reads, a common phenomenon attributable to the sequencing chemistry.</p>
<p>Post-filtering and trimming, the histogram demonstrates a marked
improvement in the quality profile across the read length. The initial
high-quality scores are maintained, yet the previously observed decline
towards the read ends is significantly mitigated. For instance, where
the pre-filtering scores for sample mock1_S96_L001_R1_001 plummeted
below a Phred score of 20 after the 250 bp mark, the post-filtering
scores show a more gradual decline, maintaining a Phred score above 20
for a more extended range. This suggests that the bases with lower
quality scores have been successfully excised from the sequence
data.</p>
<p>This improvement in the quality score distribution is indicative of
the efficacy of the trimming process, specifically targeting the removal
of low-quality bases which are often found at the 3’ ends of the reads.
Moreover, the maintenance of high-quality scores throughout the initial
and mid-regions of the reads post-trimming implies that the data
retained is of high quality and suitable for downstream bioinformatics
analyses. It is noteworthy that the stringent criteria applied during
the trimming process, such as the trimming of the 5’ end to a Phred
score of 35 and the 3’ end to a Phred score of 30, have not only
enhanced the overall quality but also have done so while preserving the
sequence length as much as possible, as evidenced by the post-trimming
average read lengths. The procedural approach, thus, ensures the removal
of bases that could otherwise compromise the reliability of subsequent
analyses due to poor quality while safeguarding the integrity and
utility of the sequencing data.</p>
</div>
<div id="status-checks-1" class="section level2">
<h2>Status Checks</h2>
<p><img src="Output/TrimmedQC/Figures/trimmed_fastqc-status-check-heatmap.jpeg" width="100%" /></p>
<p><strong>Figure 4:</strong> Post-Trimming FastQC Quality Control
Status Heatmap. This heatmap represents the outcomes of various quality
control checks performed on sequencing data after a stringent trimming
process. The columns correspond to nine mock samples, labeled
mock1_S96_R1_trimmed through mock9_S48_R1_trimmed, processed to remove
low-quality bases and adapter sequences. Each row represents a distinct
quality control metric assessed by FastQC, including evaluations of
sequence quality, GC content, N content, sequence length distribution,
duplication levels, and adapter presence. Colors indicate the status of
each metric: green for pass, yellow for warn, and red for fail. The
heatmap conveys the improvement in data quality post-processing, with an
increase in the number of passed metrics, suggesting that the trimming
protocol has effectively enhanced the quality of the sequencing reads.
Metrics such as ‘Per Base Sequence Quality’ and ‘Adapter Content’
predominantly show passes across all samples, indicating the successful
removal of low-quality ends and adapter sequences.</p>
<p>The FastQC Status Checks heatmaps provide an at-a-glance assessment
of sequencing data quality across various metrics, both before and after
quality filtering and trimming. In the pre-trimming heatmap, there is a
discernible presence of warnings (yellow) and failures (red) across
several quality metrics. For instance, the ‘Per Base Sequence Content’
and ‘Sequence Length Distribution’ metrics display a pattern of warnings
for multiple samples such as ‘mock2_S96_L001_R1_00’, suggesting
irregularities in the nucleotide composition at certain positions and
inhomogeneities in read lengths, respectively.</p>
<p>Post-trimming, there is a notable shift towards an increased number
of passes (green) across most metrics. This improvement is particularly
evident in the ‘Per Base Sequence Quality’ metric, where samples like
‘mock1_S96_R1_trimmed’ show a transition from a mix of warnings and
failures to predominantly passes. This change reflects the effectiveness
of trimming in removing low-quality bases that are often found at the
read ends, thereby enhancing the overall quality of the sequencing
data.</p>
<p>Furthermore, the ‘Adapter Content’ metric, which had displayed
warnings in several samples prior to trimming, shows clear improvement
with most samples now passing this check. This suggests that adapter
sequences have been successfully removed, reducing the potential for
artifacts in subsequent analyses. The impact of the quality filtering
and trimming procedures is thus demonstrated by an increased uniformity
in the data quality, as evidenced by the reduction in warnings and
failures. This homogenization of data quality post-trimming is
indicative of a dataset that is more consistent and reliable for
downstream bioinformatic applications, emphasizing the importance of
preprocessing in high-throughput sequencing workflows.</p>
</div>
</div>
<div id="dada2-pipeline" class="section level1">
<h1>7. DADA2 Pipeline</h1>
<p>The DADA2 pipeline represents a critical workflow for high-resolution
analysis of microbial communities using amplicon sequence data. It
encompasses a series of steps designed to model and correct
Illumina-sequenced amplicon errors, distinguish between real biological
sequences and artifacts, and infer the composition of the microbial
community. This process begins with quality filtering and trimming to
ensure high-confidence base calls, followed by dereplication and the
inference of Amplicon Sequence Variants (ASVs). Subsequent steps involve
removing chimeric sequences, which can confound taxonomic assignment,
and constructing a comprehensive ASV table. The pipeline concludes with
taxonomic classification, providing a detailed profile of microbial
diversity. The DADA2 pipeline’s systematic approach allows for a more
accurate representation of sequence diversity and abundance,
facilitating robust downstream ecological and evolutionary analyses.</p>
<div id="dada2-pipeline-setup" class="section level2">
<h2>DADA2 Pipeline Setup</h2>
<p>Before running the DADA2 pipeline, it’s crucial to set up the
environment and ensure all necessary packages are loaded. DADA2 is a
comprehensive pipeline for detecting and correcting data errors in
high-throughput sequencing datasets. The following steps outline the
initial setup required for running DADA2 in R.</p>
<pre class="r"><code># Change to the R output directory where you will run R commands for the DADA2 pipeline:
setwd(&quot;/proj/liu-compute-2023-21/users/x_morja/Microbiome_Project/16S_Part1/Routput&quot;)

# Activate R and load the dada2 package
library(dada2)
packageVersion(&quot;dada2&quot;)

# Setting a seed ensures that our results are reproducible
set.seed(123)

# Define the path to the directory containing the sequence files
path &lt;- &quot;/proj/liu-compute-2023-21/users/x_morja/Microbiome_Project/16S_Part1/trimmed_data&quot;
list.files(path)

# Organize the filenames for forward and reverse reads
fnFs &lt;- sort(list.files(path, pattern=&quot;_R1_trimmed2.fastq.gz&quot;, full.names = TRUE))
fnRs &lt;- sort(list.files(path, pattern=&quot;_R2_trimmed2.fastq.gz&quot;, full.names = TRUE))

# Extract sample names based on the filenames
sample.names &lt;- sapply(strsplit(basename(fnFs), &quot;_&quot;), `[`, 1)</code></pre>
</div>
<div id="quality-filtering-and-trimming-1" class="section level2">
<h2>Quality Filtering and Trimming</h2>
<p>After setting up the initial parameters for the DADA2 pipeline, the
next step is to perform quality filtering and trimming. This process
involves removing low-quality sequences and trimming sequences to remove
potential errors. The <code>filterAndTrim</code> function from the DADA2
package is used for this purpose.</p>
<pre class="r"><code># Set the path to the &#39;filtered&#39; directory within &#39;trimmed_data&#39;
filt_path &lt;- file.path(path, &quot;filtered&quot;)

# Create the &#39;filtered&#39; directory if it does not exist
if (!dir.exists(filt_path)) {
  dir.create(filt_path)
}

# Define the paths for the filtered forward and reverse read files
filtFs &lt;- file.path(filt_path, paste0(sample.names, &quot;_F_filt.fastq.gz&quot;))
filtRs &lt;- file.path(filt_path, paste0(sample.names, &quot;_R_filt.fastq.gz&quot;))

# Name the filtered files after the sample names for easy reference
names(filtFs) &lt;- sample.names
names(filtRs) &lt;- sample.names

# Execute the filtering and trimming process, removing PhiX contamination and compressing the output
filtered_out &lt;- filterAndTrim(fnFs, filtFs, fnRs, filtRs, rm.phix=TRUE, compress=TRUE, verbose=TRUE)

# Display the number of sequences retained after filtering
head(filtered_out)

# Convert matrix to data frame for plotting
filtered_out_df &lt;- as.data.frame(filtered_out)

# Write the data frame to a text file
write.table(filtered_out_df, file = &quot;filtered_out.txt&quot;, sep = &quot;\t&quot;, row.names = TRUE, col.names = NA, quote = FALSE)</code></pre>
</div>
<div id="error-rates" class="section level2">
<h2>Error Rates</h2>
<p>Understanding and modeling the error rates of sequencing runs is a
critical component of the DADA2 pipeline. This process allows for more
accurate inference of the true biological sequences present in the
sample. The <code>learnErrors</code> function is applied to both forward
and reverse reads to learn the error rates, which are then plotted for
visual inspection.</p>
<pre class="r"><code># Learn error rates for forward reads
errF &lt;- learnErrors(filtFs, multithread=TRUE) 

# Learn error rates for reverse reads
errR &lt;- learnErrors(filtRs, multithread=TRUE)

# Plot the estimated error rates for forward reads and save to a JPEG with high quality
jpeg(&quot;error_plot_F.jpeg&quot;, width=800, height=800, quality=100)
plotErrors(errF, nominalQ=TRUE)
dev.off()

# Plot the estimated error rates for reverse reads and save to a JPEG with high quality
jpeg(&quot;error_plot_R.jpeg&quot;, width=800, height=800, quality=100)
plotErrors(errR, nominalQ=TRUE)
dev.off()</code></pre>
</div>
<div id="dereplication-and-sample-inference" class="section level2">
<h2>Dereplication and Sample Inference</h2>
<p>Dereplication is a process that consolidates identical sequences into
a single data entry with a count of its occurrences. This step reduces
computational load and allows for more efficient subsequent analysis.
The DADA2 pipeline performs dereplication on both forward and reverse
reads before inferring sample composition.</p>
<pre class="r"><code># Dereplicate the filtered sequences for forward reads
derepFs &lt;- derepFastq(filtFs, verbose=TRUE) # For forward reads

# Dereplicate the filtered sequences for reverse reads
derepRs &lt;- derepFastq(filtRs, verbose=TRUE) # For reverse reads

# Assign names to the dereplicated data based on sample names
names(derepFs) &lt;- sample.names
names(derepRs) &lt;- sample.names</code></pre>
</div>
<div id="inferring-asvs" class="section level2">
<h2>Inferring ASVs</h2>
<p>The inference of ASVs is a critical step in the DADA2 pipeline, where
each unique sequence variant within the dataset is identified. This
process uses error rates previously calculated to distinguish between
true biological sequences and those that are errors. The code below
demonstrates how ASVs are inferred for both forward and reverse
reads.</p>
<pre class="r"><code># Inferring ASVs for forward reads
dadaFs &lt;- dada(derepFs, err=errF, multithread=TRUE, pool=FALSE)

# Inferring ASVs for reverse reads
dadaRs &lt;- dada(derepRs, err=errR, multithread=TRUE, pool=FALSE)

# Inspect the first inferred ASVs for forward and reverse reads
dadaFs[[1]]
dadaRs[[1]]

# Print out ASV inference results for forward reads
for(i in seq_along(dadaFs)) {
  cat(&quot;Sample&quot;, names(dadaFs[i]), &quot;- Forward\n&quot;)
  print(dadaFs[[i]])
  cat(&quot;\n&quot;)
}

# Print out ASV inference results for reverse reads
for(i in seq_along(dadaRs)) {
  cat(&quot;Sample&quot;, names(dadaRs[i]), &quot;- Reverse\n&quot;)
  print(dadaRs[[i]])
  cat(&quot;\n&quot;)
}</code></pre>
</div>
<div id="merging-paired-reads" class="section level2">
<h2>Merging Paired Reads</h2>
<p>Merging paired-end reads is an essential step in the analysis of
amplicon data. This process combines the forward and reverse reads for
each sequence, which have been previously inferred as ASVs, to create a
full representation of the amplicon. The <code>mergePairs</code>
function from the DADA2 package is used for this purpose, and the
resulting merged sequences are outputted as individual CSV files for
each sample.</p>
<pre class="r"><code># Merging paired reads with DADA2
mergers &lt;- mergePairs(dadaFs, derepFs, dadaRs, derepRs, trimOverhang=TRUE, verbose=TRUE)

# Write merged sequences to CSV files for each sample
for(i in seq_along(mergers)) {
  # Construct the file name based on the sample name
  file_name &lt;- paste0(&quot;Sample&quot;, names(mergers[i]), &quot;_merged_sequences.csv&quot;)
  
  # Write the merged sequences to a CSV file
  write.csv(mergers[[i]], file=file_name)
}</code></pre>
</div>
<div id="constructing-the-asv-table" class="section level2">
<h2>Constructing the ASV Table</h2>
<p>The construction of the Amplicon Sequence Variant (ASV) table is a
pivotal step in amplicon sequencing data analysis. The
<code>makeSequenceTable</code> function from the DADA2 package compiles
the merged reads into a single table, facilitating the examination of
sequence variants across all samples. Post-construction, the
distribution of sequence lengths is inspected to ensure they fall within
the expected range for targeted amplicon sequences.</p>
<pre class="r"><code># Construct the ASV table from merged sequences
seqtab &lt;- makeSequenceTable(mergers)
dim(seqtab) # View dimensions of the sequence table

# Inspect the distribution of sequence lengths
table(nchar(getSequences(seqtab)))

# Save the distribution of sequence lengths to a text file
write.table(table(nchar(getSequences(seqtab))), &quot;seqlength_distribution.txt&quot;, sep=&quot;\t&quot;)

# Plot the distribution of sequence lengths
require(ggplot2)
length_table &lt;- as.data.frame(table(nchar(colnames(seqtab))))
colnames(length_table) &lt;- c(&quot;LENGTH&quot;, &quot;COUNT&quot;)
p1 &lt;- ggplot(length_table, aes(x=LENGTH, y=COUNT)) +
  geom_bar(stat=&quot;identity&quot;) +
  ggtitle(&quot;Distribution of Sequence Lengths&quot;) +
  theme_bw() +
  theme(axis.text.x=element_text(angle=90, hjust=1)) +
  theme(axis.text.y=element_text(size=12))

# Save the plot as a high-quality JPEG with a width of 1200 pixels and a height of 1200 pixels
jpeg(&quot;sequence_length_distribution.jpeg&quot;, width = 1200, height = 1200, quality = 100)
print(p1 + scale_y_log10()) # Log scale for better visualization
dev.off()

# Filter out sequences outside the expected length range
seqtab2 &lt;- seqtab[,nchar(colnames(seqtab)) %in% seq(300, 460)]

# Reinspect the distribution of sequence lengths after filtering
length_table2 &lt;- as.data.frame(table(nchar(colnames(seqtab2))))
colnames(length_table2) &lt;- c(&quot;LENGTH&quot;, &quot;COUNT&quot;)
p2 &lt;- ggplot(length_table2, aes(x=LENGTH, y=COUNT)) +
  geom_bar(stat=&quot;identity&quot;) +
  ggtitle(&quot;Filtered Distribution of Sequence Lengths&quot;) +
  theme_bw() +
  theme(axis.text.x=element_text(angle=90, hjust=1)) +
  theme(axis.text.y=element_text(size=12))

# Save the filtered plot as a high-quality JPEG with a width of 1200 pixels and a height of 1200 pixels
jpeg(&quot;filtered_sequence_length_distribution.jpeg&quot;, width = 1200, height = 1200, quality = 100)
print(p2 + scale_y_log10()) # Log scale for better visualization
dev.off()</code></pre>
</div>
<div id="chimera-identification-and-removal" class="section level2">
<h2>Chimera Identification and Removal</h2>
<p>Chimera identification is a critical quality control step in the
processing of 16S rRNA gene sequencing data. Chimeric sequences, which
are artifacts of PCR amplification, can lead to the overestimation of
diversity if not properly identified and removed. The
<code>removeBimeraDenovo</code> function in the DADA2 package is used
for this purpose, employing a consensus approach that compares sequences
within the dataset to identify chimeras.</p>
<pre class="r"><code># Remove chimeric sequences from the ASV table
seqtab.nochim &lt;- removeBimeraDenovo(seqtab, method=&quot;consensus&quot;, multithread=TRUE, verbose=TRUE)
dim(seqtab.nochim) # View dimensions of the non-chimeric sequence table

# Calculate and print the proportion of sequences remaining after chimera removal
non_chimeric_proportion &lt;- sum(seqtab.nochim) / sum(seqtab)
cat(&quot;Proportion of non-chimeric sequences:&quot;, non_chimeric_proportion, &quot;\n&quot;)

# Estimate the total initial abundance before chimera removal
total_initial_abundance &lt;- sum(seqtab) / (1 - non_chimeric_proportion)

# Calculate and print the abundance of sequences lost to chimera removal
abundance_lost &lt;- total_initial_abundance - sum(seqtab.nochim)
cat(&quot;Number of sequences lost to chimera removal:&quot;, total_initial_abundance - sum(seqtab.nochim), &quot;\n&quot;)
cat(&quot;Total abundance of lost sequences:&quot;, abundance_lost, &quot;\n&quot;)</code></pre>
</div>
<div id="tracking-reads-through-the-pipeline" class="section level2">
<h2>Tracking Reads Through the Pipeline</h2>
<p>Tracking the number of reads at each stage of the DADA2 pipeline is
essential for understanding the effects of quality filtering, denoising,
merging, and chimera removal on the dataset. This process helps to
ensure that the data quality is maintained throughout the pipeline and
provides a transparent view of the data processing workflow.</p>
<pre class="r"><code># Define a function to sum the unique reads for a given sample
getN &lt;- function(x) sum(getUniques(x))

# Create a summary table to track the read counts through the pipeline stages
summary_tab &lt;- data.frame(
  row.names = sample.names,
  dada2_input = filtered_out[,1],
  filtered = filtered_out[,2],
  denoisedF = sapply(dadaFs, getN),
  denoisedR = sapply(dadaRs, getN),
  merged = sapply(mergers, getN),
  nonchim = rowSums(seqtab.nochim),
  final_perc_reads_retained = round(rowSums(seqtab.nochim) / filtered_out[,1] * 100, 1)
)

# Write the summary table to a text file
write.table(summary_tab, &quot;summary_tab_pseudo.txt&quot;, sep=&quot;\t&quot;)

# Write the summary table to a TSV file for easy sharing and further analysis
write.table(summary_tab, &quot;read-count-tracking.tsv&quot;, quote=FALSE, sep=&quot;\t&quot;, col.names=NA)</code></pre>
</div>
<div id="taxonomy-assignment" class="section level2">
<h2>Taxonomy Assignment</h2>
<p>Taxonomy assignment is a critical step in analyzing amplicon sequence
data. It involves matching sequences to known reference sequences to
identify the taxonomic lineage of each amplicon sequence variant (ASV).
The <code>assignTaxonomy</code> function from the DADA2 package is used
for this purpose, utilizing a reference training set to classify
sequences to the genus level. The <code>addSpecies</code> function
further refines this classification by adding species-level
identification when possible.</p>
<pre class="r"><code># Assign taxonomy using the DADA2 package and the Silva reference database
taxa &lt;- assignTaxonomy(seqtab.nochim, &quot;/proj/liu-compute-2023-21/16S_amplicon/RefSeq_silva/silva_nr99_v138.1_train_set.fa.gz&quot;, multithread=TRUE, tryRC=TRUE, minBoot=80)

# Add species-level identification to the assigned taxonomy
taxa &lt;- addSpecies(taxa, &quot;/proj/liu-compute-2023-21/16S_amplicon/RefSeq_silva/silva_nr99_v138.1_wSpecies_train_set.fa.gz&quot;, allowMultiple=TRUE)

# Inspect the first few entries of the taxonomy assignment
taxa.print &lt;- taxa # Copy the taxa object for display purposes
rownames(taxa.print) &lt;- NULL # Remove sequence rownames for a cleaner display
head(taxa.print)</code></pre>
</div>
<div id="data-retrieval" class="section level2">
<h2>Data Retrieval</h2>
<p>After the completion of the DADA2 pipeline and taxonomy assignment,
the final step is to organize and retrieve the data for further
analysis. This involves renaming the sequence headers to a more
manageable format and creating count and taxonomy tables for the
ASVs.</p>
<pre class="r"><code># Renaming sequence headers to a more manageable format (ASV_1, ASV_2, ...)
asv_seqs &lt;- colnames(seqtab.nochim)
asv_headers &lt;- vector(&quot;character&quot;, length(asv_seqs))

for (i in seq_along(asv_seqs)) {
  asv_headers[i] &lt;- paste(&quot;&gt;ASV&quot;, i, sep=&quot;_&quot;)
}

# Creating a count table for ASVs
asv_tab &lt;- t(seqtab.nochim)
row.names(asv_tab) &lt;- sub(&quot;&gt;&quot;, &quot;&quot;, asv_headers)
write.table(asv_tab, &quot;ASVs_counts.txt&quot;, sep=&quot;\t&quot;, quote=FALSE, col.names=NA)

# Creating a taxonomy table for ASVs
asv_tax &lt;- taxa
row.names(asv_tax) &lt;- sub(&quot;&gt;&quot;, &quot;&quot;, asv_headers)
write.table(asv_tax, &quot;ASVs_taxonomy.txt&quot;, sep=&quot;\t&quot;, quote=FALSE, col.names=NA)</code></pre>
</div>
</div>
<div id="dada2-results-and-interpretation" class="section level1">
<h1>8. DADA2 Results and Interpretation</h1>
<p>This section provides an overview of the results obtained from the
DADA2 pipeline. The DADA2 process includes several steps: quality
filtering, dereplication, sample inference, merging paired reads,
removing chimeras, and assigning taxonomy. Each step is designed to
refine the dataset and enhance the quality of the sequence data, which
is critical for accurate microbiome analysis.</p>
<div id="quality-filtering-and-trimming-results" class="section level2">
<h2>Quality Filtering and Trimming Results</h2>
<p><strong>Table 3:</strong> This table displays the number of sequence
reads retained after the quality filtering and trimming step of the
DADA2 pipeline. The ‘reads.in’ column shows the total number of reads
input into the filtering process, while the ‘reads.out’ column shows the
number of high-quality reads retained after filtering. This step is
crucial for ensuring that subsequent analyses are based on accurate and
reliable sequence data.</p>
<table class="table table-striped table-hover table-condensed table-responsive" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;">
Samples
</th>
<th style="text-align:center;">
reads.in
</th>
<th style="text-align:center;">
reads.out
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
mock1_S96_R1_trimmed2.fastq.gz
</td>
<td style="text-align:center;">
30512
</td>
<td style="text-align:center;">
30506
</td>
</tr>
<tr>
<td style="text-align:center;">
mock2_S96_R1_trimmed2.fastq.gz
</td>
<td style="text-align:center;">
114074
</td>
<td style="text-align:center;">
114061
</td>
</tr>
<tr>
<td style="text-align:center;">
mock3_S69_R1_trimmed2.fastq.gz
</td>
<td style="text-align:center;">
171514
</td>
<td style="text-align:center;">
171513
</td>
</tr>
<tr>
<td style="text-align:center;">
mock4_S96_R1_trimmed2.fastq.gz
</td>
<td style="text-align:center;">
112936
</td>
<td style="text-align:center;">
112927
</td>
</tr>
<tr>
<td style="text-align:center;">
mock5_S54_R1_trimmed2.fastq.gz
</td>
<td style="text-align:center;">
35745
</td>
<td style="text-align:center;">
35744
</td>
</tr>
<tr>
<td style="text-align:center;">
mock6_S46_R1_trimmed2.fastq.gz
</td>
<td style="text-align:center;">
111537
</td>
<td style="text-align:center;">
111536
</td>
</tr>
<tr>
<td style="text-align:center;">
mock7_S46_R1_trimmed2.fastq.gz
</td>
<td style="text-align:center;">
228448
</td>
<td style="text-align:center;">
228448
</td>
</tr>
<tr>
<td style="text-align:center;">
mock8_S33_R1_trimmed2.fastq.gz
</td>
<td style="text-align:center;">
71603
</td>
<td style="text-align:center;">
71602
</td>
</tr>
<tr>
<td style="text-align:center;">
mock9_S48_R1_trimmed2.fastq.gz
</td>
<td style="text-align:center;">
83980
</td>
<td style="text-align:center;">
83948
</td>
</tr>
</tbody>
</table>
<p>The data presented in the table above reflect the number of
sequencing reads retained after the application of a filtering and
trimming process aimed at removing PhiX contamination—a common control
in Illumina sequencing runs. This step is crucial to prepare the
sequencing data for accurate downstream analyses by excluding sequences
that do not originate from the target microbial community.</p>
<p>For example, in the case of
<code>mock1_S96_R1_trimmed2.fastq.gz</code>, the filtering process has
resulted in the retention of 30,506 reads out of 30,512 input reads,
indicating that only a minimal number of reads were discarded. This
trend of retaining the vast majority of reads is consistent across all
samples. Notably, <code>mock7_S46_R1_trimmed2.fastq.gz</code>
demonstrates no loss of reads, with all 228,448 reads passing the
filtering criteria. Such high retention rates suggest that the PhiX
contamination was either very low or the filter parameters were
well-optimized to preserve the maximum number of legitimate reads.</p>
<p>The minimal difference between the ‘reads.in’ and ‘reads.out’ columns
across samples signifies the filtering criteria’s stringency and
effectiveness. It implies that the majority of the sequence data is of
high quality and that the microbial sequences are largely free from the
specific contaminant targeted by the filtering process. This high
retention rate after the filtering stage is indicative of a successful
preprocessing step, ensuring that the subsequent stages of data
analysis, such as taxonomic classification or diversity estimation, are
based on authentic and relevant sequencing data. The consistency in read
retention across samples, such as
<code>mock6_S46_R1_trimmed2.fastq.gz</code> losing only one read,
further supports the reliability of the filtering process and the
quality of the input data.</p>
</div>
<div id="error-rates-results" class="section level2">
<h2>Error Rates Results</h2>
<p><img src="Microbiome-Project_files/figure-html/figure5-1.png" width="100%" />
<strong>Figure 5:</strong> Error Rate Estimation for Forward and Reverse
Reads in DADA2 Pipeline Analysis. Panel A displays the estimated error
rates for forward reads, while Panel B illustrates those for reverse
reads. In both panels, the x-axis represents the consensus quality
score, and the y-axis shows the corresponding error frequency (log10
scale). Black dots indicate observed error rates at each consensus
quality score, and the red line represents the error rates estimated by
the DADA2 model. Error rates are shown for all possible nucleotide
transitions, with each plot corresponding to a specific transition type,
as labeled (e.g., A2A, A2C, etc.). The trend across the plots
illustrates the expected inverse relationship between quality scores and
error rates. Notably, the model exhibits a close fit to observed data in
most transitions, with some exceptions such as T2G in reverse reads,
where deviations from the model are apparent. These plots are crucial
for assessing the accuracy of sequence data and ensuring the reliability
of downstream analyses such as taxonomic classification and diversity
estimation.</p>
<p>The figure above displays the estimated error rates for both forward
(labelled as A) and reverse (labelled as B) reads, which are part of a
DADA2 pipeline analysis. This analysis is commonly used for processing
16S rRNA gene sequences. In each plot within the larger figure, the
error rates are plotted against consensus quality scores. Each
sub-figure represents a different sample or nucleotide transition (e.g.,
A to C, G to T), as indicated by the labels such as A2A, A2C, etc. The
observed error rates at each quality score are depicted by black dots,
and the red line represents the model’s estimated error rates.</p>
<p>Observations from plot A (Forward Read Error Rates):</p>
<ol style="list-style-type: decimal">
<li>A general trend can be observed where the error rates decrease as
consensus quality scores increase, which aligns with the expected
behavior in sequencing where higher quality scores are associated with
lower error rates.</li>
<li>The model seems to fit the observed data well for most transitions,
as the red line closely follows the distribution of the black dots.</li>
<li>Certain transitions, notably G2G, show a pronounced decline in error
rates at lower quality scores, indicating these transitions are reliably
predicted across a range of quality scores.</li>
</ol>
<p>Observations from plot B (Reverse Read Error Rates):</p>
<ol style="list-style-type: decimal">
<li>Similar to the forward reads, the reverse read error rates also
decrease as the consensus quality scores rise.</li>
<li>There are outliers, particularly noticeable in the T2G transition,
where the observed error rates do not align with the model’s estimates.
These deviations could be attributed to technical sequencing errors,
variations specific to the sample, or limitations in the model’s
assumptions.</li>
<li>The reverse read error plots tend to show a steeper decrease in
error rates as quality scores increase, especially evident in
transitions like C2C and T2C, when compared to the forward read
plots.</li>
</ol>
<p>In both sub-figures, outliers that do not follow the main trend may
point to potential issues with specific reads or suggest that for some
transitions, the quality scores do not consistently predict error
rates.</p>
</div>
<div id="constructing-the-asv-table-1" class="section level2">
<h2>Constructing the ASV Table</h2>
<p><img src="Microbiome-Project_files/figure-html/figure6-1.png" width="100%" />
<strong>Figure 6:</strong> Distribution of sequence lengths in the
sequence table from merged DNA sequences. A) The initial distribution of
sequence lengths exhibits extensive variability with multiple peaks,
suggesting a diverse set of sequences. B) After filtering to include
only sequences within a specified length range, the distribution
displays more pronounced peaks and reduced variability, indicating a
refined dataset for subsequent analysis. Sequence counts are displayed
on a logarithmic scale to highlight variations across a wide range of
values.</p>
<p>Graph A illustrates the initial distribution of sequence lengths,
revealing a broad range of lengths with notable variability. The
sequence counts are plotted on a logarithmic scale to accommodate a wide
distribution of values, facilitating the visualization of data spanning
several orders of magnitude. This scale transformation is evident by the
vertical axis being labelled in powers of ten, suggesting that the
sequence count varies exponentially with sequence length. Certain
lengths appear to be more prevalent than others, as evidenced by the
peaks in the distribution, which may indicate common biological sequence
lengths or artifacts of the sequencing process.</p>
<p>Graph B shows the distribution after applying a filter to include
only sequences within a specific length range, presumably the range of
300 to 460 nucleotides as indicated by the code. This restriction has
resulted in a more focused distribution, eliminating the sequences
outside the defined range and potentially improving the quality of the
data for downstream analysis. The peaks within this graph are sharper
and more defined compared to Graph A, reflecting a more uniform
distribution within the selected length range.</p>
<div id="tracking-reads-through-the-pipeline-1" class="section level3">
<h3>Tracking Reads Through the Pipeline</h3>
<p><strong>Table 4:</strong> Sequential Read Counts at Key Stages of the
DADA2 Pipeline for Quality Control of Metabarcoding Data. This table
presents the read counts for nine mock samples at critical phases of the
DADA2 pipeline, including quality filtering, denoising, merging, and
chimera removal. It starts with initial read counts (dada2_input),
followed by post-filtering (filtered), denoising (denoisedF and
denoisedR), merging (merged), and concludes with non-chimeric reads
(nonchim). The final column quantifies the retention of reads
(final_perc_reads_retained) after processing. This table serves to
evaluate the pipeline’s efficacy in preserving usable sequence data
through successive stages. Variability in read retention across samples
highlights the impact of bioinformatics workflow on data quality and
integrity.”</p>
<table class="table table-striped table-hover table-condensed table-responsive" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;">
Samples
</th>
<th style="text-align:center;">
dada2_input
</th>
<th style="text-align:center;">
filtered
</th>
<th style="text-align:center;">
denoisedF
</th>
<th style="text-align:center;">
denoisedR
</th>
<th style="text-align:center;">
merged
</th>
<th style="text-align:center;">
nonchim
</th>
<th style="text-align:center;">
final_perc_reads_retained
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
mock1
</td>
<td style="text-align:center;">
30512
</td>
<td style="text-align:center;">
30506
</td>
<td style="text-align:center;">
29959
</td>
<td style="text-align:center;">
30211
</td>
<td style="text-align:center;">
18276
</td>
<td style="text-align:center;">
18186
</td>
<td style="text-align:center;">
59.6
</td>
</tr>
<tr>
<td style="text-align:center;">
mock2
</td>
<td style="text-align:center;">
114074
</td>
<td style="text-align:center;">
114061
</td>
<td style="text-align:center;">
113143
</td>
<td style="text-align:center;">
113493
</td>
<td style="text-align:center;">
88560
</td>
<td style="text-align:center;">
83520
</td>
<td style="text-align:center;">
73.2
</td>
</tr>
<tr>
<td style="text-align:center;">
mock3
</td>
<td style="text-align:center;">
171514
</td>
<td style="text-align:center;">
171513
</td>
<td style="text-align:center;">
170626
</td>
<td style="text-align:center;">
170759
</td>
<td style="text-align:center;">
141313
</td>
<td style="text-align:center;">
131703
</td>
<td style="text-align:center;">
76.8
</td>
</tr>
<tr>
<td style="text-align:center;">
mock4
</td>
<td style="text-align:center;">
112936
</td>
<td style="text-align:center;">
112927
</td>
<td style="text-align:center;">
111890
</td>
<td style="text-align:center;">
112409
</td>
<td style="text-align:center;">
87389
</td>
<td style="text-align:center;">
82468
</td>
<td style="text-align:center;">
73.0
</td>
</tr>
<tr>
<td style="text-align:center;">
mock5
</td>
<td style="text-align:center;">
35745
</td>
<td style="text-align:center;">
35744
</td>
<td style="text-align:center;">
35335
</td>
<td style="text-align:center;">
35403
</td>
<td style="text-align:center;">
31954
</td>
<td style="text-align:center;">
31374
</td>
<td style="text-align:center;">
87.8
</td>
</tr>
<tr>
<td style="text-align:center;">
mock6
</td>
<td style="text-align:center;">
111537
</td>
<td style="text-align:center;">
111536
</td>
<td style="text-align:center;">
110750
</td>
<td style="text-align:center;">
111091
</td>
<td style="text-align:center;">
48033
</td>
<td style="text-align:center;">
46918
</td>
<td style="text-align:center;">
42.1
</td>
</tr>
<tr>
<td style="text-align:center;">
mock7
</td>
<td style="text-align:center;">
228448
</td>
<td style="text-align:center;">
228448
</td>
<td style="text-align:center;">
227349
</td>
<td style="text-align:center;">
227772
</td>
<td style="text-align:center;">
188587
</td>
<td style="text-align:center;">
181921
</td>
<td style="text-align:center;">
79.6
</td>
</tr>
<tr>
<td style="text-align:center;">
mock8
</td>
<td style="text-align:center;">
71603
</td>
<td style="text-align:center;">
71602
</td>
<td style="text-align:center;">
70692
</td>
<td style="text-align:center;">
71243
</td>
<td style="text-align:center;">
52618
</td>
<td style="text-align:center;">
51149
</td>
<td style="text-align:center;">
71.4
</td>
</tr>
<tr>
<td style="text-align:center;">
mock9
</td>
<td style="text-align:center;">
83980
</td>
<td style="text-align:center;">
83948
</td>
<td style="text-align:center;">
83562
</td>
<td style="text-align:center;">
83597
</td>
<td style="text-align:center;">
5207
</td>
<td style="text-align:center;">
5203
</td>
<td style="text-align:center;">
6.2
</td>
</tr>
</tbody>
</table>
<p>The table above enumerates read counts at each key processing stage:
the initial input into DADA2 (<code>dada2_input</code>), post-filtering
(<code>filtered</code>), following denoising of forward
(<code>denoisedF</code>) and reverse (<code>denoisedR</code>) reads,
after merging the reads (<code>merged</code>), post-chimera removal
(<code>nonchim</code>), and the final percentage of reads retained
(<code>final_perc_reads_retained</code>) relative to the initial input
count.</p>
<p>For example, sample mock1 is documented to commence with 30,512
reads, from which 18,186 remain as non-chimeric, constituting 59.6% of
the starting read count. Conversely, sample mock5 displays a retention
rate of 87.8%, with 31,374 out of the initial 35,745 reads surviving as
non-chimeric sequences. In stark contrast, sample mock9 reveals a marked
decline in the retention of reads, with a mere 6.2% (5,203 reads) of the
starting 83,980 reads preserved post-processing, signaling a significant
attrition during the stages of the pipeline, potentially attributable to
factors such as a high incidence of chimeric sequences or suboptimal
initial read quality.</p>
<p>The read retention metrics presented in Table 1 are pivotal for
evaluating the DADA2 pipeline’s effectiveness and the rigor applied
during the bioinformatics processing. The differential retention rates
of reads among samples underscore the unique characteristics each sample
may exhibit through the sequencing and subsequent processing phases.
Divergence in sample handling, sequencing coverage, and intrinsic
biological diversity can influence the proportion of chimeric sequences
and the success of the merge operation. Monitoring read counts through
the pipeline is indispensable for comprehending the influence of the
data processing workflow on the integrity and fidelity of the final
sequence dataset.</p>
</div>
</div>
<div id="data-retrieval-1" class="section level2">
<h2>Data Retrieval</h2>
<p><strong>Table 5:</strong> The ASVs count table is derived from a
denoised sequence table devoid of chimeric sequences. Each entry in the
count table corresponds to an ASV, which is a unique sequence variant
that has been identified within the sample set. The rows of this table
represent the ASVs, indexed in a simplified format such as ‘ASV_1’,
‘ASV_2’, etc., to facilitate ease of reference. The columns correspond
to the various samples in which these ASVs were detected. The cells of
the table contain integer counts representing the number of times each
ASV was observed in each sample.”</p>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; overflow-x: scroll; width:100%; ">
<table class="table table-striped table-hover table-condensed table-responsive" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
ASVs
</th>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
mock1
</th>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
mock2
</th>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
mock3
</th>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
mock4
</th>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
mock5
</th>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
mock6
</th>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
mock7
</th>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
mock8
</th>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
mock9
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
ASV_1
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
25057
</td>
<td style="text-align:center;">
37512
</td>
<td style="text-align:center;">
23636
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
59397
</td>
<td style="text-align:center;">
14101
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_2
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
15749
</td>
<td style="text-align:center;">
24826
</td>
<td style="text-align:center;">
15572
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
37403
</td>
<td style="text-align:center;">
10143
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_3
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
13654
</td>
<td style="text-align:center;">
21280
</td>
<td style="text-align:center;">
14599
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
34961
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_4
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
18039
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
8371
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_5
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
3199
</td>
<td style="text-align:center;">
4893
</td>
<td style="text-align:center;">
3533
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
3250
</td>
<td style="text-align:center;">
6658
</td>
<td style="text-align:center;">
2138
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_6
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
5753
</td>
<td style="text-align:center;">
10757
</td>
<td style="text-align:center;">
3622
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_7
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
5466
</td>
<td style="text-align:center;">
8068
</td>
<td style="text-align:center;">
5154
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_8
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
4755
</td>
<td style="text-align:center;">
7364
</td>
<td style="text-align:center;">
4967
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_9
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
5049
</td>
<td style="text-align:center;">
7148
</td>
<td style="text-align:center;">
4754
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_10
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
3526
</td>
<td style="text-align:center;">
3677
</td>
<td style="text-align:center;">
3434
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
4713
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_11
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
3113
</td>
<td style="text-align:center;">
4202
</td>
<td style="text-align:center;">
2911
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
2283
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_12
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
1330
</td>
<td style="text-align:center;">
2259
</td>
<td style="text-align:center;">
1132
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
1627
</td>
<td style="text-align:center;">
2786
</td>
<td style="text-align:center;">
1067
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_13
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
8354
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_14
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
8210
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_15
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
2674
</td>
<td style="text-align:center;">
4903
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_16
</td>
<td style="text-align:center;">
6604
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_17
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
1456
</td>
<td style="text-align:center;">
2290
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
1452
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
873
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_18
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
798
</td>
<td style="text-align:center;">
1237
</td>
<td style="text-align:center;">
717
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
912
</td>
<td style="text-align:center;">
1820
</td>
<td style="text-align:center;">
465
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_19
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
5727
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_20
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
5257
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_21
</td>
<td style="text-align:center;">
3716
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_22
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
479
</td>
<td style="text-align:center;">
525
</td>
<td style="text-align:center;">
362
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
504
</td>
<td style="text-align:center;">
1016
</td>
<td style="text-align:center;">
467
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_23
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
940
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
2288
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_24
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
911
</td>
<td style="text-align:center;">
1436
</td>
<td style="text-align:center;">
838
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_25
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
393
</td>
<td style="text-align:center;">
458
</td>
<td style="text-align:center;">
376
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
455
</td>
<td style="text-align:center;">
910
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_26
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
712
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
719
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
560
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
498
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_27
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
2227
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_28
</td>
<td style="text-align:center;">
2180
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_29
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
894
</td>
<td style="text-align:center;">
929
</td>
<td style="text-align:center;">
303
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_30
</td>
<td style="text-align:center;">
1864
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_31
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
1763
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_32
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
1612
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_33
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
1550
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_34
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
1322
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_35
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
1263
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_36
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
1144
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_37
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
362
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
274
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
478
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_38
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
1092
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_39
</td>
<td style="text-align:center;">
1067
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_40
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
147
</td>
<td style="text-align:center;">
918
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_41
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
1055
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_42
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
1033
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_43
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
1022
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_44
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
1003
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_45
</td>
<td style="text-align:center;">
913
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_46
</td>
<td style="text-align:center;">
907
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_47
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
70
</td>
<td style="text-align:center;">
789
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_48
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
813
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_49
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
193
</td>
<td style="text-align:center;">
353
</td>
<td style="text-align:center;">
225
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_50
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
758
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_51
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
342
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
339
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_52
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
643
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_53
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
630
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_54
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
628
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_55
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
600
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_56
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
382
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
200
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_57
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
534
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_58
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
268
</td>
<td style="text-align:center;">
265
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_59
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
520
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_60
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
514
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_61
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
335
</td>
<td style="text-align:center;">
163
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_62
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
133
</td>
<td style="text-align:center;">
90
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
270
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_63
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
472
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_64
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
459
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_65
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
433
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_66
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
202
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
212
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_67
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
404
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_68
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
362
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_69
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
356
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_70
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
333
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_71
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
329
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_72
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
318
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_73
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
309
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_74
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
303
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_75
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
296
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_76
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
292
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_77
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
289
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_78
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
66
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
213
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_79
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
273
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_80
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
266
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_81
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
264
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_82
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
256
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_83
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
253
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_84
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
249
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_85
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
247
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_86
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
242
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_87
</td>
<td style="text-align:center;">
229
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_88
</td>
<td style="text-align:center;">
229
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_89
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
69
</td>
<td style="text-align:center;">
149
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_90
</td>
<td style="text-align:center;">
209
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_91
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
209
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_92
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
204
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_93
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
195
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_94
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
193
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_95
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
174
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_96
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
161
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_97
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
152
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_98
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
149
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_99
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
142
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_100
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
134
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_101
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
133
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_102
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
131
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_103
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
130
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_104
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
130
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_105
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
127
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_106
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
126
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_107
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
126
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_108
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
119
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_109
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
119
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_110
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
116
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_111
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
114
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_112
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
110
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_113
</td>
<td style="text-align:center;">
103
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_114
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
102
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_115
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
99
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_116
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
86
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_117
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
78
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_118
</td>
<td style="text-align:center;">
75
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_119
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
73
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_120
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
72
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_121
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
71
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_122
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
66
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_123
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
65
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_124
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
65
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_125
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
61
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_126
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
61
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_127
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
57
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_128
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
57
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_129
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
54
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_130
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
44
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_131
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
43
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_132
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
6
</td>
<td style="text-align:center;">
34
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_133
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
35
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_134
</td>
<td style="text-align:center;">
34
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_135
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
33
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_136
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
32
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_137
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
32
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_138
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
31
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_139
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
31
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_140
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
31
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_141
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
30
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_142
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
29
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_143
</td>
<td style="text-align:center;">
28
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_144
</td>
<td style="text-align:center;">
28
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_145
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
28
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_146
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
28
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_147
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
28
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_148
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
27
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_149
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
24
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_150
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
22
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_151
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
21
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_152
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
19
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_153
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
18
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_154
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
17
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_155
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
16
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_156
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
16
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_157
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_158
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_159
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_160
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
8
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_161
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
5
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_162
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
5
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_163
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
4
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_164
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
3
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_165
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
2
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_166
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
2
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_167
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
2
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_168
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
2
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_169
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
2
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_170
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
2
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_171
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
2
</td>
</tr>
</tbody>
</table>
</div>
<p>The ASVs count table above presents the abundance of each Amplicon
Sequence Variant (ASV) across a collection of mock samples. For example,
<code>ASV_1</code> is most abundant in <code>mock7</code> with 59,397
counts, indicating a high presence of this particular microbial sequence
in that sample. Conversely, <code>ASV_4</code> is exclusively found in
<code>mock6</code> and <code>mock8</code>, with 18,039 and 8,371 counts
respectively, suggesting a more restricted distribution.</p>
<p><strong>Table 6:</strong> The ASVs taxonomy table provides the
taxonomic classification for each ASV identified in the study. Similar
to the count table, each row represents an ASV, labeled in an analogous
simplified format for consistency and ease of cross-referencing. The
columns of this table delineate the hierarchical taxonomic ranks at
which each ASV has been classified, typically ranging from kingdom down
to genus or species, depending on the resolution of the
classification.</p>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; overflow-x: scroll; width:100%; ">
<table class="table table-striped table-hover table-condensed table-responsive" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
ASVs
</th>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
Kingdom
</th>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
Phylum
</th>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
Class
</th>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
Order
</th>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
Family
</th>
<th style="text-align:center;position: sticky; top:0; background-color: #FFFFFF;">
Genus
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
ASV_1
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Bacillales
</td>
<td style="text-align:center;">
Bacillaceae
</td>
<td style="text-align:center;">
Bacillus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_2
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Moraxellaceae
</td>
<td style="text-align:center;">
Acinetobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_3
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Enterobacterales
</td>
<td style="text-align:center;">
Enterobacteriaceae
</td>
<td style="text-align:center;">
Escherichia-Shigella
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_4
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Enterobacterales
</td>
<td style="text-align:center;">
Enterobacteriaceae
</td>
<td style="text-align:center;">
Escherichia-Shigella
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_5
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Clostridia
</td>
<td style="text-align:center;">
Clostridiales
</td>
<td style="text-align:center;">
Clostridiaceae
</td>
<td style="text-align:center;">
Clostridium sensu stricto 1
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_6
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Burkholderiales
</td>
<td style="text-align:center;">
Neisseriaceae
</td>
<td style="text-align:center;">
Neisseria
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_7
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Burkholderiales
</td>
<td style="text-align:center;">
Neisseriaceae
</td>
<td style="text-align:center;">
Neisseria
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_8
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Bacteroidaceae
</td>
<td style="text-align:center;">
Bacteroides
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_9
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Bacteroidaceae
</td>
<td style="text-align:center;">
Bacteroides
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_10
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Alphaproteobacteria
</td>
<td style="text-align:center;">
Rhodobacterales
</td>
<td style="text-align:center;">
Rhodobacteraceae
</td>
<td style="text-align:center;">
Rhodobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_11
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Porphyromonadaceae
</td>
<td style="text-align:center;">
Porphyromonas
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_12
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Campylobacterota
</td>
<td style="text-align:center;">
Campylobacteria
</td>
<td style="text-align:center;">
Campylobacterales
</td>
<td style="text-align:center;">
Helicobacteraceae
</td>
<td style="text-align:center;">
Helicobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_13
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Bacillales
</td>
<td style="text-align:center;">
Bacillaceae
</td>
<td style="text-align:center;">
Bacillus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_14
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Deinococcota
</td>
<td style="text-align:center;">
Deinococci
</td>
<td style="text-align:center;">
Deinococcales
</td>
<td style="text-align:center;">
Deinococcaceae
</td>
<td style="text-align:center;">
Deinococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_15
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Porphyromonadaceae
</td>
<td style="text-align:center;">
Porphyromonas
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_16
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Bacillales
</td>
<td style="text-align:center;">
Bacillaceae
</td>
<td style="text-align:center;">
Bacillus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_17
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Lactobacillales
</td>
<td style="text-align:center;">
Streptococcaceae
</td>
<td style="text-align:center;">
Streptococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_18
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Lactobacillales
</td>
<td style="text-align:center;">
Enterococcaceae
</td>
<td style="text-align:center;">
Enterococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_19
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Enterobacterales
</td>
<td style="text-align:center;">
Enterobacteriaceae
</td>
<td style="text-align:center;">
Escherichia-Shigella
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_20
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Moraxellaceae
</td>
<td style="text-align:center;">
Acinetobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_21
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Enterobacterales
</td>
<td style="text-align:center;">
Enterobacteriaceae
</td>
<td style="text-align:center;">
Escherichia-Shigella
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_22
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Staphylococcales
</td>
<td style="text-align:center;">
Staphylococcaceae
</td>
<td style="text-align:center;">
Staphylococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_23
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Porphyromonadaceae
</td>
<td style="text-align:center;">
Porphyromonas
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_24
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Bacteroidaceae
</td>
<td style="text-align:center;">
Bacteroides
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_25
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Staphylococcales
</td>
<td style="text-align:center;">
Staphylococcaceae
</td>
<td style="text-align:center;">
Staphylococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_26
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Lactobacillales
</td>
<td style="text-align:center;">
Lactobacillaceae
</td>
<td style="text-align:center;">
Lactobacillus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_27
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Alphaproteobacteria
</td>
<td style="text-align:center;">
Rhodobacterales
</td>
<td style="text-align:center;">
Rhodobacteraceae
</td>
<td style="text-align:center;">
Rhodobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_28
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Bacteroidaceae
</td>
<td style="text-align:center;">
Bacteroides
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_29
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Bacteroidaceae
</td>
<td style="text-align:center;">
Bacteroides
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_30
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Pseudomonadaceae
</td>
<td style="text-align:center;">
Pseudomonas
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_31
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Burkholderiales
</td>
<td style="text-align:center;">
Neisseriaceae
</td>
<td style="text-align:center;">
Neisseria
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_32
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Bacteroidaceae
</td>
<td style="text-align:center;">
Bacteroides
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_33
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Campylobacterota
</td>
<td style="text-align:center;">
Campylobacteria
</td>
<td style="text-align:center;">
Campylobacterales
</td>
<td style="text-align:center;">
Helicobacteraceae
</td>
<td style="text-align:center;">
Helicobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_34
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Bacteroidaceae
</td>
<td style="text-align:center;">
Bacteroides
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_35
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Lactobacillales
</td>
<td style="text-align:center;">
Streptococcaceae
</td>
<td style="text-align:center;">
Streptococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_36
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Moraxellaceae
</td>
<td style="text-align:center;">
Acinetobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_37
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Lactobacillales
</td>
<td style="text-align:center;">
Streptococcaceae
</td>
<td style="text-align:center;">
Streptococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_38
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Clostridia
</td>
<td style="text-align:center;">
Clostridiales
</td>
<td style="text-align:center;">
Clostridiaceae
</td>
<td style="text-align:center;">
Clostridium sensu stricto 1
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_39
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Lactobacillales
</td>
<td style="text-align:center;">
Streptococcaceae
</td>
<td style="text-align:center;">
Streptococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_40
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Bacteroidaceae
</td>
<td style="text-align:center;">
Bacteroides
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_41
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Pseudomonadaceae
</td>
<td style="text-align:center;">
Pseudomonas
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_42
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Alphaproteobacteria
</td>
<td style="text-align:center;">
Rhodobacterales
</td>
<td style="text-align:center;">
Rhodobacteraceae
</td>
<td style="text-align:center;">
Rhodobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_43
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Moraxellaceae
</td>
<td style="text-align:center;">
Acinetobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_44
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Bacillales
</td>
<td style="text-align:center;">
Bacillaceae
</td>
<td style="text-align:center;">
Bacillus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_45
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Clostridia
</td>
<td style="text-align:center;">
Clostridiales
</td>
<td style="text-align:center;">
Clostridiaceae
</td>
<td style="text-align:center;">
Clostridium sensu stricto 1
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_46
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Porphyromonadaceae
</td>
<td style="text-align:center;">
Porphyromonas
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_47
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Burkholderiales
</td>
<td style="text-align:center;">
Neisseriaceae
</td>
<td style="text-align:center;">
Neisseria
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_48
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Deinococcota
</td>
<td style="text-align:center;">
Deinococci
</td>
<td style="text-align:center;">
Deinococcales
</td>
<td style="text-align:center;">
Deinococcaceae
</td>
<td style="text-align:center;">
Deinococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_49
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Bacteroidaceae
</td>
<td style="text-align:center;">
Bacteroides
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_50
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Bacillales
</td>
<td style="text-align:center;">
Bacillaceae
</td>
<td style="text-align:center;">
Bacillus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_51
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Moraxellaceae
</td>
<td style="text-align:center;">
Acinetobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_52
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Pseudomonadaceae
</td>
<td style="text-align:center;">
Pseudomonas
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_53
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Bacillales
</td>
<td style="text-align:center;">
Bacillaceae
</td>
<td style="text-align:center;">
Bacillus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_54
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Lactobacillales
</td>
<td style="text-align:center;">
Enterococcaceae
</td>
<td style="text-align:center;">
Enterococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_55
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Campylobacterota
</td>
<td style="text-align:center;">
Campylobacteria
</td>
<td style="text-align:center;">
Campylobacterales
</td>
<td style="text-align:center;">
Helicobacteraceae
</td>
<td style="text-align:center;">
Helicobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_56
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Pseudomonadaceae
</td>
<td style="text-align:center;">
Pseudomonas
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_57
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Moraxellaceae
</td>
<td style="text-align:center;">
Acinetobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_58
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Burkholderiales
</td>
<td style="text-align:center;">
Neisseriaceae
</td>
<td style="text-align:center;">
Neisseria
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_59
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Enterobacterales
</td>
<td style="text-align:center;">
Enterobacteriaceae
</td>
<td style="text-align:center;">
Escherichia-Shigella
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_60
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Pseudomonadaceae
</td>
<td style="text-align:center;">
Pseudomonas
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_61
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Porphyromonadaceae
</td>
<td style="text-align:center;">
Porphyromonas
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_62
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Alphaproteobacteria
</td>
<td style="text-align:center;">
Rhodobacterales
</td>
<td style="text-align:center;">
Rhodobacteraceae
</td>
<td style="text-align:center;">
Rhodobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_63
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Bacillales
</td>
<td style="text-align:center;">
Bacillaceae
</td>
<td style="text-align:center;">
Bacillus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_64
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Moraxellaceae
</td>
<td style="text-align:center;">
Acinetobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_65
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Lactobacillales
</td>
<td style="text-align:center;">
Streptococcaceae
</td>
<td style="text-align:center;">
Streptococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_66
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Moraxellaceae
</td>
<td style="text-align:center;">
Acinetobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_67
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Bacillales
</td>
<td style="text-align:center;">
Bacillaceae
</td>
<td style="text-align:center;">
Bacillus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_68
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Moraxellaceae
</td>
<td style="text-align:center;">
Acinetobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_69
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Pseudomonadaceae
</td>
<td style="text-align:center;">
Pseudomonas
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_70
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Deinococcota
</td>
<td style="text-align:center;">
Deinococci
</td>
<td style="text-align:center;">
Deinococcales
</td>
<td style="text-align:center;">
Deinococcaceae
</td>
<td style="text-align:center;">
Deinococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_71
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Bacteroidaceae
</td>
<td style="text-align:center;">
Bacteroides
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_72
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Campylobacterota
</td>
<td style="text-align:center;">
Campylobacteria
</td>
<td style="text-align:center;">
Campylobacterales
</td>
<td style="text-align:center;">
Helicobacteraceae
</td>
<td style="text-align:center;">
Helicobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_73
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Bacillales
</td>
<td style="text-align:center;">
Bacillaceae
</td>
<td style="text-align:center;">
Bacillus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_74
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Moraxellaceae
</td>
<td style="text-align:center;">
Acinetobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_75
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Enterobacterales
</td>
<td style="text-align:center;">
Enterobacteriaceae
</td>
<td style="text-align:center;">
Escherichia-Shigella
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_76
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Bacteroidaceae
</td>
<td style="text-align:center;">
Bacteroides
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_77
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Lactobacillales
</td>
<td style="text-align:center;">
Enterococcaceae
</td>
<td style="text-align:center;">
Enterococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_78
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Campylobacterota
</td>
<td style="text-align:center;">
Campylobacteria
</td>
<td style="text-align:center;">
Campylobacterales
</td>
<td style="text-align:center;">
Helicobacteraceae
</td>
<td style="text-align:center;">
Helicobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_79
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Deinococcota
</td>
<td style="text-align:center;">
Deinococci
</td>
<td style="text-align:center;">
Deinococcales
</td>
<td style="text-align:center;">
Deinococcaceae
</td>
<td style="text-align:center;">
Deinococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_80
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Bacteroidaceae
</td>
<td style="text-align:center;">
Bacteroides
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_81
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Enterobacterales
</td>
<td style="text-align:center;">
Enterobacteriaceae
</td>
<td style="text-align:center;">
Escherichia-Shigella
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_82
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Deinococcota
</td>
<td style="text-align:center;">
Deinococci
</td>
<td style="text-align:center;">
Deinococcales
</td>
<td style="text-align:center;">
Deinococcaceae
</td>
<td style="text-align:center;">
Deinococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_83
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Pseudomonadaceae
</td>
<td style="text-align:center;">
Pseudomonas
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_84
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Moraxellaceae
</td>
<td style="text-align:center;">
Acinetobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_85
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Enterobacterales
</td>
<td style="text-align:center;">
Enterobacteriaceae
</td>
<td style="text-align:center;">
Escherichia-Shigella
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_86
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Lactobacillales
</td>
<td style="text-align:center;">
Lactobacillaceae
</td>
<td style="text-align:center;">
Lactobacillus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_87
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Staphylococcales
</td>
<td style="text-align:center;">
Staphylococcaceae
</td>
<td style="text-align:center;">
Staphylococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_88
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Lactobacillales
</td>
<td style="text-align:center;">
Lactobacillaceae
</td>
<td style="text-align:center;">
Lactobacillus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_89
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Porphyromonadaceae
</td>
<td style="text-align:center;">
Porphyromonas
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_90
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Lactobacillales
</td>
<td style="text-align:center;">
Enterococcaceae
</td>
<td style="text-align:center;">
Enterococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_91
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Lactobacillales
</td>
<td style="text-align:center;">
Streptococcaceae
</td>
<td style="text-align:center;">
Streptococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_92
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Alphaproteobacteria
</td>
<td style="text-align:center;">
Rhodobacterales
</td>
<td style="text-align:center;">
Rhodobacteraceae
</td>
<td style="text-align:center;">
Rhodobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_93
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Deinococcota
</td>
<td style="text-align:center;">
Deinococci
</td>
<td style="text-align:center;">
Deinococcales
</td>
<td style="text-align:center;">
Deinococcaceae
</td>
<td style="text-align:center;">
Deinococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_94
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Lactobacillales
</td>
<td style="text-align:center;">
Streptococcaceae
</td>
<td style="text-align:center;">
Streptococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_95
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Bacteroidaceae
</td>
<td style="text-align:center;">
Bacteroides
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_96
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Bacteroidaceae
</td>
<td style="text-align:center;">
Bacteroides
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_97
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Staphylococcales
</td>
<td style="text-align:center;">
Staphylococcaceae
</td>
<td style="text-align:center;">
Staphylococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_98
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Pseudomonadaceae
</td>
<td style="text-align:center;">
Pseudomonas
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_99
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Alphaproteobacteria
</td>
<td style="text-align:center;">
Rhodobacterales
</td>
<td style="text-align:center;">
Rhodobacteraceae
</td>
<td style="text-align:center;">
Rhodobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_100
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Staphylococcales
</td>
<td style="text-align:center;">
Staphylococcaceae
</td>
<td style="text-align:center;">
Staphylococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_101
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Enterobacterales
</td>
<td style="text-align:center;">
Enterobacteriaceae
</td>
<td style="text-align:center;">
Escherichia-Shigella
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_102
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Enterobacterales
</td>
<td style="text-align:center;">
Enterobacteriaceae
</td>
<td style="text-align:center;">
Escherichia-Shigella
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_103
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Bacillales
</td>
<td style="text-align:center;">
Bacillaceae
</td>
<td style="text-align:center;">
Bacillus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_104
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Lactobacillales
</td>
<td style="text-align:center;">
Streptococcaceae
</td>
<td style="text-align:center;">
Streptococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_105
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Bacillales
</td>
<td style="text-align:center;">
Bacillaceae
</td>
<td style="text-align:center;">
Bacillus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_106
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Burkholderiales
</td>
<td style="text-align:center;">
Neisseriaceae
</td>
<td style="text-align:center;">
Neisseria
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_107
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Pseudomonadaceae
</td>
<td style="text-align:center;">
Pseudomonas
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_108
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Pseudomonadaceae
</td>
<td style="text-align:center;">
Pseudomonas
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_109
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Porphyromonadaceae
</td>
<td style="text-align:center;">
Porphyromonas
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_110
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Pseudomonadaceae
</td>
<td style="text-align:center;">
Pseudomonas
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_111
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Deinococcota
</td>
<td style="text-align:center;">
Deinococci
</td>
<td style="text-align:center;">
Deinococcales
</td>
<td style="text-align:center;">
Deinococcaceae
</td>
<td style="text-align:center;">
Deinococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_112
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Campylobacterota
</td>
<td style="text-align:center;">
Campylobacteria
</td>
<td style="text-align:center;">
Campylobacterales
</td>
<td style="text-align:center;">
Helicobacteraceae
</td>
<td style="text-align:center;">
Helicobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_113
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Moraxellaceae
</td>
<td style="text-align:center;">
Acinetobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_114
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Bacillales
</td>
<td style="text-align:center;">
Bacillaceae
</td>
<td style="text-align:center;">
Bacillus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_115
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Enterobacterales
</td>
<td style="text-align:center;">
Enterobacteriaceae
</td>
<td style="text-align:center;">
Escherichia-Shigella
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_116
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Bacillales
</td>
<td style="text-align:center;">
Bacillaceae
</td>
<td style="text-align:center;">
Bacillus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_117
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Campylobacterota
</td>
<td style="text-align:center;">
Campylobacteria
</td>
<td style="text-align:center;">
Campylobacterales
</td>
<td style="text-align:center;">
Helicobacteraceae
</td>
<td style="text-align:center;">
Helicobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_118
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Burkholderiales
</td>
<td style="text-align:center;">
Neisseriaceae
</td>
<td style="text-align:center;">
Neisseria
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_119
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Pseudomonadaceae
</td>
<td style="text-align:center;">
Pseudomonas
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_120
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Clostridia
</td>
<td style="text-align:center;">
Clostridiales
</td>
<td style="text-align:center;">
Clostridiaceae
</td>
<td style="text-align:center;">
Clostridium sensu stricto 1
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_121
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Clostridia
</td>
<td style="text-align:center;">
Clostridiales
</td>
<td style="text-align:center;">
Clostridiaceae
</td>
<td style="text-align:center;">
Clostridium sensu stricto 1
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_122
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Deinococcota
</td>
<td style="text-align:center;">
Deinococci
</td>
<td style="text-align:center;">
Deinococcales
</td>
<td style="text-align:center;">
Deinococcaceae
</td>
<td style="text-align:center;">
Deinococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_123
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Burkholderiales
</td>
<td style="text-align:center;">
Neisseriaceae
</td>
<td style="text-align:center;">
Neisseria
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_124
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Staphylococcales
</td>
<td style="text-align:center;">
Staphylococcaceae
</td>
<td style="text-align:center;">
Staphylococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_125
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Bacteroidaceae
</td>
<td style="text-align:center;">
Bacteroides
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_126
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Campylobacterota
</td>
<td style="text-align:center;">
Campylobacteria
</td>
<td style="text-align:center;">
Campylobacterales
</td>
<td style="text-align:center;">
Helicobacteraceae
</td>
<td style="text-align:center;">
Helicobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_127
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Bacteroidaceae
</td>
<td style="text-align:center;">
Bacteroides
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_128
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Porphyromonadaceae
</td>
<td style="text-align:center;">
Porphyromonas
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_129
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
NA
</td>
<td style="text-align:center;">
NA
</td>
<td style="text-align:center;">
NA
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_130
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Moraxellaceae
</td>
<td style="text-align:center;">
Acinetobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_131
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Bacillales
</td>
<td style="text-align:center;">
Bacillaceae
</td>
<td style="text-align:center;">
Bacillus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_132
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Bacteroidaceae
</td>
<td style="text-align:center;">
Bacteroides
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_133
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Burkholderiales
</td>
<td style="text-align:center;">
Neisseriaceae
</td>
<td style="text-align:center;">
Neisseria
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_134
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Bacillales
</td>
<td style="text-align:center;">
Bacillaceae
</td>
<td style="text-align:center;">
Bacillus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_135
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Bacteroidaceae
</td>
<td style="text-align:center;">
Bacteroides
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_136
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
NA
</td>
<td style="text-align:center;">
NA
</td>
<td style="text-align:center;">
NA
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_137
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Alphaproteobacteria
</td>
<td style="text-align:center;">
Rhodobacterales
</td>
<td style="text-align:center;">
Rhodobacteraceae
</td>
<td style="text-align:center;">
Rhodobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_138
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
NA
</td>
<td style="text-align:center;">
NA
</td>
<td style="text-align:center;">
NA
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_139
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Moraxellaceae
</td>
<td style="text-align:center;">
Acinetobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_140
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Bacteroidaceae
</td>
<td style="text-align:center;">
Bacteroides
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_141
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Actinobacteriota
</td>
<td style="text-align:center;">
Actinobacteria
</td>
<td style="text-align:center;">
Propionibacteriales
</td>
<td style="text-align:center;">
Propionibacteriaceae
</td>
<td style="text-align:center;">
Cutibacterium
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_142
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
NA
</td>
<td style="text-align:center;">
NA
</td>
<td style="text-align:center;">
NA
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_143
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Campylobacterota
</td>
<td style="text-align:center;">
Campylobacteria
</td>
<td style="text-align:center;">
Campylobacterales
</td>
<td style="text-align:center;">
Helicobacteraceae
</td>
<td style="text-align:center;">
Helicobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_144
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Moraxellaceae
</td>
<td style="text-align:center;">
Acinetobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_145
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Campylobacterota
</td>
<td style="text-align:center;">
Campylobacteria
</td>
<td style="text-align:center;">
Campylobacterales
</td>
<td style="text-align:center;">
Helicobacteraceae
</td>
<td style="text-align:center;">
Helicobacter
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_146
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Bacteroidaceae
</td>
<td style="text-align:center;">
Bacteroides
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_147
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Deinococcota
</td>
<td style="text-align:center;">
Deinococci
</td>
<td style="text-align:center;">
Deinococcales
</td>
<td style="text-align:center;">
Deinococcaceae
</td>
<td style="text-align:center;">
Deinococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_148
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Lactobacillales
</td>
<td style="text-align:center;">
Streptococcaceae
</td>
<td style="text-align:center;">
Streptococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_149
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Negativicutes
</td>
<td style="text-align:center;">
Veillonellales-Selenomonadales
</td>
<td style="text-align:center;">
Veillonellaceae
</td>
<td style="text-align:center;">
Veillonella
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_150
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Bacteroidaceae
</td>
<td style="text-align:center;">
Bacteroides
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_151
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Bacteroidaceae
</td>
<td style="text-align:center;">
Bacteroides
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_152
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Actinobacteriota
</td>
<td style="text-align:center;">
Actinobacteria
</td>
<td style="text-align:center;">
Propionibacteriales
</td>
<td style="text-align:center;">
Propionibacteriaceae
</td>
<td style="text-align:center;">
Cutibacterium
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_153
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Negativicutes
</td>
<td style="text-align:center;">
Veillonellales-Selenomonadales
</td>
<td style="text-align:center;">
Veillonellaceae
</td>
<td style="text-align:center;">
Veillonella
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_154
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Bacteroidaceae
</td>
<td style="text-align:center;">
Bacteroides
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_155
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Bacteroidaceae
</td>
<td style="text-align:center;">
Bacteroides
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_156
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
NA
</td>
<td style="text-align:center;">
NA
</td>
<td style="text-align:center;">
NA
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_157
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Deinococcota
</td>
<td style="text-align:center;">
Deinococci
</td>
<td style="text-align:center;">
Deinococcales
</td>
<td style="text-align:center;">
Deinococcaceae
</td>
<td style="text-align:center;">
Deinococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_158
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
NA
</td>
<td style="text-align:center;">
NA
</td>
<td style="text-align:center;">
NA
</td>
<td style="text-align:center;">
NA
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_159
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
NA
</td>
<td style="text-align:center;">
NA
</td>
<td style="text-align:center;">
NA
</td>
<td style="text-align:center;">
NA
</td>
<td style="text-align:center;">
NA
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_160
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Deinococcota
</td>
<td style="text-align:center;">
Deinococci
</td>
<td style="text-align:center;">
Deinococcales
</td>
<td style="text-align:center;">
Deinococcaceae
</td>
<td style="text-align:center;">
Deinococcus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_161
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Bacteroidaceae
</td>
<td style="text-align:center;">
Bacteroides
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_162
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
Bacillales
</td>
<td style="text-align:center;">
Bacillaceae
</td>
<td style="text-align:center;">
Bacillus
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_163
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
NA
</td>
<td style="text-align:center;">
NA
</td>
<td style="text-align:center;">
NA
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_164
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Bacteroidaceae
</td>
<td style="text-align:center;">
Bacteroides
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_165
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Pseudomonadaceae
</td>
<td style="text-align:center;">
Pseudomonas
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_166
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Pseudomonadaceae
</td>
<td style="text-align:center;">
Pseudomonas
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_167
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Pseudomonadales
</td>
<td style="text-align:center;">
Pseudomonadaceae
</td>
<td style="text-align:center;">
Pseudomonas
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_168
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Bacteroidaceae
</td>
<td style="text-align:center;">
Bacteroides
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_169
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Firmicutes
</td>
<td style="text-align:center;">
Bacilli
</td>
<td style="text-align:center;">
NA
</td>
<td style="text-align:center;">
NA
</td>
<td style="text-align:center;">
NA
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_170
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Bacteroidota
</td>
<td style="text-align:center;">
Bacteroidia
</td>
<td style="text-align:center;">
Bacteroidales
</td>
<td style="text-align:center;">
Porphyromonadaceae
</td>
<td style="text-align:center;">
Porphyromonas
</td>
</tr>
<tr>
<td style="text-align:center;">
ASV_171
</td>
<td style="text-align:center;">
Bacteria
</td>
<td style="text-align:center;">
Proteobacteria
</td>
<td style="text-align:center;">
Gammaproteobacteria
</td>
<td style="text-align:center;">
Enterobacterales
</td>
<td style="text-align:center;">
Enterobacteriaceae
</td>
<td style="text-align:center;">
Escherichia-Shigella
</td>
</tr>
</tbody>
</table>
</div>
<p>The ASV taxonomy table associates each ASV with its corresponding
taxonomic classification. Taking <code>ASV_1</code> as an instance, it
is classified within the kingdom Bacteria, phylum Firmicutes, class
Bacilli, order Bacillales, family Bacillaceae, and genus Bacillus. This
level of taxonomic resolution provides insight into the potential
microbial composition and diversity within the samples. Similarly,
<code>ASV_2</code> and <code>ASV_3</code> are both categorized under the
kingdom Bacteria and phylum Proteobacteria, but they are distinguished
at the family and genus levels, being Moraxellaceae (genus
Acinetobacter) and Enterobacteriaceae (genus Escherichia-Shigella)
respectively.</p>
</div>
</div>
<div id="questions-and-answers" class="section level1">
<h1>9. Questions and Answers</h1>
<p><span
style="color: red; font-weight: bold; font-size: 18px;">Question
1:</span> <strong>Did all the samples pass the quality control? Are
there any modules that failed (which ones)? Is there a biological
explanation for why they failed?</strong></p>
<p><span
style="color: green; font-weight: bold; font-size: 18px;">Answer:</span>
The analysis of the quality control metrics suggests that not all
samples passed the quality control checks uniformly. Specifically, ‘Per
Base N Content’ and ‘Sequence Length Distribution’ have displayed
warnings and failures in certain samples, such as ‘mock2_S96_L001_R1_00’
and ‘mock3_S69_L001_R1_00’. These issues may indicate suboptimal base
call accuracy and unexpected variations in read lengths, which could
potentially impact the integrity of the sequencing data.</p>
<p>Furthermore, the duplication rates are notably high across all
samples, as seen in the summary statistics with ‘mock1_S96_L001_R1_001’
exhibiting a duplication rate of 93.8%. While high duplication rates can
result from technical factors such as PCR amplification biases, they may
also reflect biological realities, such as a low diversity of sequences
present in the samples due to the dominance of a few microbial species
or the presence of highly conserved regions across different taxa.</p>
<p>The sequence quality histograms and status checks heatmap provide a
granular view of the quality across samples. While the initial sequence
quality is high, as evidenced by Phred scores consistently above 30, the
decline in quality towards the end of the reads suggests that trimming
may be necessary to remove low-quality bases that could otherwise
confound the analysis. The ‘Per Sequence GC Content’ metric generally
passed across all samples, indicating that the GC representation is
balanced and not indicative of contamination or sequencing bias.</p>
<p>In conclusion, while the overall quality of the sequencing run is
high, certain quality metrics failed for some samples, warranting
further investigation. The biological explanations for these failures
could be related to the nature of the microbial communities sampled, the
presence of conserved sequences, or the technicalities of the sequencing
process itself. Before proceeding, it is imperative to address these
quality control issues to ensure the accuracy of downstream
analyses.</p>
<p><span
style="color: red; font-weight: bold; font-size: 18px;">Question
2:</span> <strong>Which trimming settings did you use and
why?</strong></p>
<p><span
style="color: green; font-weight: bold; font-size: 18px;">answer:</span>
The trimming settings applied to the sequence data were defined to
selectively retain high-quality sequences for downstream analysis. The
parameters included trimming from the 3’ end of the reads, with a Phred
quality score threshold set at 30. Bases below this quality score were
removed, and reads were subsequently excluded from the analysis if their
length fell below 100 bases after trimming. The application of these
parameters is a conventional practice intended to reduce the inclusion
of erroneous base calls, which is crucial for the reliability of
subsequent bioinformatic evaluations.</p>
<p><span
style="color: red; font-weight: bold; font-size: 18px;">Question
3:</span> <strong>Keep track of reads lost on that process: how many (N)
input read you had and how many (N and %) output reads you have? How
many (N and %) of reads were removed with the trimming?</strong></p>
<p><span
style="color: green; font-weight: bold; font-size: 18px;">answer:</span>
Upon review of the statistical data provided for several samples, a
summative evaluation can be drawn. Initially, the project handled a
combined total of 2,505,112 reads across all samples before the trimming
process was initiated. Following the completion of the trimming
protocol, the remaining reads across all samples totaled 2,495,896. This
retention rate indicates that the trimming protocol preserved 99.63% of
the input reads, which is a measure of the protocol’s selectivity for
quality while maintaining the integrity of the dataset.</p>
<p>The breakdown of reads lost during the trimming process across the
provided samples is as follows:</p>
<ul>
<li>For the sample starting with 335,528 reads, trimming was responsible
for the loss of 1,006 reads, which translates to a marginal attrition
rate of 0.30% from the original read count.</li>
<li>The sample initially comprising 482,950 reads saw a reduction by
2,406 reads post-trimming, marking a 0.50% decrease.</li>
<li>From the starting count of 340,860 reads in another sample, a
slightly higher loss was registered at 1,152 reads, equating to a 0.34%
loss rate.</li>
<li>A similar trend was observed in the sample with 375,410 initial
reads, where trimming accounted for a loss of 1,230 reads, or
0.33%.</li>
<li>The largest sample, with 718,238 reads, experienced a trimming loss
of 2,912 reads, corresponding to a 0.41% reduction.</li>
<li>The smallest provided sample, with an input of 252,126 reads, had
the least percentage loss of 0.20%, which resulted in the removal of 510
reads.</li>
</ul>
<p>In percentage terms, the total number of reads removed was roughly
0.37%, a comparatively low figure that underscores the precision of the
trimming algorithm in retaining high-quality sequence data for
downstream applications. The data demonstrate a high-efficiency trimming
process, where the majority of the data loss (measured by the number of
reads) hovers around a fractional half-percent mark, underscoring a
consistent and targeted approach to data quality enhancement. The
resultant dataset, therefore, reflects a high-fidelity collection of
sequence reads poised for reliable analysis in microbial community
characterization or other molecular investigations. This degree of read
retention ensures that the majority of the original sequence data is
maintained, bolstering the robustness of the biological inferences that
can be drawn from the data. The minor variance observed in the
percentage of reads removed across samples likely reflects intrinsic
differences in the quality of the raw sequences provided for each
sample, which is a normal occurrence in next-generation sequencing
workflows.</p>
<p><span
style="color: red; font-weight: bold; font-size: 18px;">Question
4:</span> <strong>Is the compromise between quality and number of reads
acceptable or should you revise the trimming and filtering?</strong></p>
<p><span
style="color: green; font-weight: bold; font-size: 18px;">answer:</span>
The trimming and quality filtering process successfully improved the
overall data quality by removing low-quality bases without significantly
impacting the sequence composition or depth of coverage. Although the
high duplication rates persisted post-trimming, the data retained is of
high quality, suitable for downstream analyses. It appears that the
compromise between quality and number of reads is acceptable, as the
trimming process effectively removed low-quality bases, as indicated by
the maintenance of high-quality scores across the reads and the improved
status check outcomes.</p>
<p>However, the persistence of high duplication rates may necessitate
further investigation into the library preparation methods or the nature
of the samples. The consistent failure rates post-trimming imply that
the remaining data is of high quality and that the stringent quality
criteria applied during the trimming process were effective. The data
refinement through these preprocessing steps ensures that subsequent
analyses are based on sequences of reliable quality, which is crucial
for microbiome research that relies on accurate sequence
information.</p>
<p>The provided information suggests that the quality filtering and
trimming processes were effective in enhancing the quality of the
sequencing data. However, before making a final decision on the
acceptability of the compromise between quality and number of reads, it
is recommended to consider the specific research goals and the impact of
high duplication rates on the intended downstream analyses. If the
analyses are sensitive to duplication rates, further optimization may be
needed. Otherwise, the current compromise may be acceptable for
proceeding with the analysis.</p>
<p><span
style="color: red; font-weight: bold; font-size: 18px;">Question
5:</span> <strong>Which version of Dada2 you worked with (why is this
informality relevant?)?</strong></p>
<p><span
style="color: green; font-weight: bold; font-size: 18px;">answer:</span>
The version of Dada2, as indicated by the R output, is
<code>1.28.0</code>. This information is relevant for several
reasons:</p>
<ul>
<li><strong>Reproducibility:</strong> Knowing the exact version of a
software package used in an analysis is crucial for reproducibility. If
someone else (or you, at a later time) wants to replicate your results,
they will need to use the same version of the software, as different
versions may have different algorithms or default settings that can lead
to different results.</li>
<li><strong>Compatibility:</strong> Some versions of Dada2 (or any
software package) may only be compatible with certain versions of R or
other R packages. If you’re sharing your work with others, they’ll need
to know which version is compatible with their setup.</li>
<li><strong>Bug Fixes and Updates:</strong> Software packages are
updated regularly to fix bugs, improve performance, and add features.
Knowing the version can help determine whether you have the latest bug
fixes and features or whether you need to update to get them.</li>
<li><strong>Methodological Rigor:</strong> When writing up methods for
publication or sharing with the scientific community, it’s important to
specify the version so that others understand the context and specifics
of your analysis. This is part of the methodological rigor expected in
scientific research.</li>
<li><strong>Support and Troubleshooting:</strong> If you encounter
issues or need help, knowing the version you’re working with is
essential for getting accurate support and troubleshooting advice,
whether from community forums, colleagues, or official support
channels.</li>
</ul>
<p><span
style="color: red; font-weight: bold; font-size: 18px;">Question
6:</span> <strong>Which type and how many sequences were
filtered?</strong></p>
<p><span
style="color: green; font-weight: bold; font-size: 18px;">answer:</span>
The filtering and trimming process was instrumental in the retention of
high-quality reads while selectively removing sequences associated with
PhiX contamination, a control element in Illumina sequencing. The
procedure demonstrated high specificity and effectiveness, as evidenced
by the negligible reduction in read counts post-filtering across the
samples. For instance, the sample
<code>mock1_S96_R1_trimmed2.fastq.gz</code> retained 30,506 of 30,512
input reads, indicative of an extremely low discard rate. The consistent
pattern of high read retention, including the sample
<code>mock7_S46_R1_trimmed2.fastq.gz</code> which reported no loss,
suggests that the PhiX contamination was minimal or that the filtering
parameters were finely tuned to maintain the integrity of legitimate
reads.</p>
<p>The total number of sequences filtered out across all samples due to
phiX contamination is 64. This is a very small fraction of the total
sequences, which suggests that the sequencing run was of high quality
with minimal contamination from the control. The fact that these numbers
are so low also indicates that the previous quality control steps were
effective in ensuring that the data moving forward is of high quality
for further analysis.</p>
<p>This stringent filtering approach underscores the quality of the
sequence data and the successful exclusion of contaminants. The
retention of such a high proportion of reads post-filtering establishes
the groundwork for subsequent analytical steps, including taxonomic
classification and diversity estimation, which rely on the authenticity
and relevance of the sequencing data. The consistency observed in read
retention, as seen with <code>mock6_S46_R1_trimmed2.fastq.gz</code>
where only a single read was lost, corroborates the robustness of the
filtering process and the caliber of the input data.</p>
<p><span
style="color: red; font-weight: bold; font-size: 18px;">Question
7:</span> <strong>Do the error rates look reasonable? Do you think you
have a good fit between the observed and estimated error rate? Do the
forward and reverse look similar?</strong></p>
<p><span
style="color: green; font-weight: bold; font-size: 18px;">answer:</span>
The error rate estimation for both forward and reverse reads, as part of
the DADA2 pipeline analysis, shows that the model closely fits the
observed data in most nucleotide transitions. This is visualized through
plots where the black dots (observed error rates) are generally in
alignment with the red line (model’s estimated error rates), supporting
the reliability of the error model used by DADA2. Notably, the inverse
relationship between quality scores and error rates, which is expected
in sequencing data, is evident across the plots.</p>
<p>For forward reads, the decline in error rates as consensus quality
scores increase is consistent with standard sequencing expectations, and
the model appears to fit well for most transitions. For example, the G2G
transition shows a pronounced decline in error rates at lower quality
scores, suggesting that these transitions are reliably predicted by the
model across various quality scores.</p>
<p>In reverse reads, a similar decreasing trend in error rates with
rising quality scores is observed. However, there are some outliers,
particularly in the T2G transition, where observed error rates do not
closely follow the model’s estimates. These discrepancies might be due
to technical sequencing errors, sample-specific variations, or
limitations in the error model.</p>
<p>Furthermore, reverse read error plots tend to show a steeper decrease
in error rates with increasing quality scores, especially in transitions
like C2C and T2C, when compared to the forward read plots. The presence
of outliers in both forward and reverse read plots suggests that there
may be issues with specific reads or that for some transitions, the
quality scores do not consistently predict error rates.</p>
<p>In conclusion, the error rates seem reasonable, and there is
generally a good fit between observed and estimated error rates,
indicating an accurate error model. While the forward and reverse reads
exhibit similar trends, the reverse reads show a steeper decrease in
error rates and some deviations in specific transitions. These findings
are essential for ensuring the accuracy of sequence data and the
reliability of downstream analyses such as taxonomic classification and
diversity estimation.</p>
<p><span
style="color: red; font-weight: bold; font-size: 18px;">Question
8:</span> <strong>How many unique sequences you had, and how many ASVs
were inferred? Is it similar between forward and reverse
reads?</strong></p>
<p><span
style="color: green; font-weight: bold; font-size: 18px;">answer:</span>
The table below summarizes the count of unique sequences and the number
of Amplicon Sequence Variants (ASVs) inferred from both forward and
reverse reads across different samples.</p>
<p><strong>Table 7:</strong> This table encapsulates the comparison of
unique sequence counts and inferred ASVs for forward and reverse reads
across different samples, illustrating the discrepancies in sequence
diversity and ASV inference between the two read directions. The data
underscore variations in the microbial community’s characterization,
suggesting the influence of read direction on microbial diversity
assessment.</p>
<table class="table table-striped table-hover table-condensed table-responsive" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;">
Sample
</th>
<th style="text-align:center;">
Unique Sequences (Forward)
</th>
<th style="text-align:center;">
ASVs (Forward)
</th>
<th style="text-align:center;">
Unique Sequences (Reverse)
</th>
<th style="text-align:center;">
ASVs (Reverse)
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
mock1
</td>
<td style="text-align:center;">
15,433
</td>
<td style="text-align:center;">
22
</td>
<td style="text-align:center;">
7,335
</td>
<td style="text-align:center;">
26
</td>
</tr>
<tr>
<td style="text-align:center;">
mock2
</td>
<td style="text-align:center;">
26,943
</td>
<td style="text-align:center;">
139
</td>
<td style="text-align:center;">
22,247
</td>
<td style="text-align:center;">
39
</td>
</tr>
<tr>
<td style="text-align:center;">
mock3
</td>
<td style="text-align:center;">
36,284
</td>
<td style="text-align:center;">
276
</td>
<td style="text-align:center;">
36,395
</td>
<td style="text-align:center;">
54
</td>
</tr>
<tr>
<td style="text-align:center;">
mock4
</td>
<td style="text-align:center;">
32,888
</td>
<td style="text-align:center;">
106
</td>
<td style="text-align:center;">
24,906
</td>
<td style="text-align:center;">
45
</td>
</tr>
<tr>
<td style="text-align:center;">
mock5
</td>
<td style="text-align:center;">
12,448
</td>
<td style="text-align:center;">
49
</td>
<td style="text-align:center;">
11,460
</td>
<td style="text-align:center;">
29
</td>
</tr>
<tr>
<td style="text-align:center;">
mock6
</td>
<td style="text-align:center;">
30,345
</td>
<td style="text-align:center;">
44
</td>
<td style="text-align:center;">
22,747
</td>
<td style="text-align:center;">
42
</td>
</tr>
<tr>
<td style="text-align:center;">
mock7
</td>
<td style="text-align:center;">
51,150
</td>
<td style="text-align:center;">
126
</td>
<td style="text-align:center;">
42,609
</td>
<td style="text-align:center;">
53
</td>
</tr>
<tr>
<td style="text-align:center;">
mock8
</td>
<td style="text-align:center;">
22,400
</td>
<td style="text-align:center;">
42
</td>
<td style="text-align:center;">
16,346
</td>
<td style="text-align:center;">
38
</td>
</tr>
<tr>
<td style="text-align:center;">
mock9
</td>
<td style="text-align:center;">
21,699
</td>
<td style="text-align:center;">
24
</td>
<td style="text-align:center;">
17,708
</td>
<td style="text-align:center;">
37
</td>
</tr>
</tbody>
</table>
<p>The analysis of sequence data from various mock samples yielded a
substantial number of unique sequences and inferred ASVs, as detailed in
the table above. For instance, sample mock1 presented with 15,433 unique
forward sequences, from which 22 ASVs were inferred, and 7,335 unique
reverse sequences, leading to the inference of 26 ASVs. This pattern of
identifying unique sequences and inferring ASVs is observable across all
samples, albeit with varying counts.</p>
<p>Upon reviewing the entire dataset, it is evident that the number of
unique sequences and the inferred ASVs are not uniform between forward
and reverse reads. Generally, the number of unique sequences identified
in the forward reads is higher compared to the reverse reads. However,
this does not consistently translate to a higher number of inferred ASVs
for forward reads; in some cases, reverse reads yield a greater number
of ASVs despite fewer unique sequences. This discrepancy suggests that
reverse reads may sometimes capture a more diverse segment of the
microbial community, or that the inference process for ASVs is
influenced by factors beyond mere sequence count, such as the quality of
reads and the error correction efficiency of the DADA2 pipeline. The
variability between forward and reverse reads underscores the importance
of considering both read directions to gain a comprehensive
understanding of microbial diversity.</p>
<p><span
style="color: red; font-weight: bold; font-size: 18px;">Question
9:</span> <strong>A minOveralp=12 is very short. What is the expected
overlap of your paired-end sequences here?</strong></p>
<p><span
style="color: green; font-weight: bold; font-size: 18px;">answer:</span>
The expected overlap for paired-end sequences can be calculated based on
the length of the reads and the size of the amplicon region they are
supposed to cover. Given the following information:</p>
<ul>
<li>Amplicon size is approximately 460 base pairs (bps).</li>
<li>Each read is 300 bps long (2x300 bp paired-end reads).</li>
<li>The primers are used to amplify the V3-V4 region, which is the part
of the 16S rRNA gene that the reads are covering.</li>
</ul>
<p>The calculation for the overlap can be done as follows:</p>
<ol style="list-style-type: decimal">
<li>Calculate the total length covered by both reads:</li>
</ol>
<p><span class="math inline">\(300 \, \text{bps (forward read)} + 300 \,
\text{bps (reverse read)} = 600 \, \text{bps}\)</span>.</p>
<ol start="2" style="list-style-type: decimal">
<li>Subtract the size of the amplicon from the total length covered by
both reads to find the overlap:</li>
</ol>
<p><span class="math inline">\(600 \, \text{bps} - 460 \, \text{bps
(amplicon size)} = 140 \, \text{bps overlap}\)</span>.</p>
<p>So, with the given read lengths and amplicon size, an overlap of
approximately 140 bps would be expected. This is well above the minimum
default requirement of 12 bps for the <code>mergePairs</code> function
in DADA2, suggesting that the default setting is conservative and the
actual overlap provides a more robust basis for merging the reads
accurately.</p>
<p><span
style="color: red; font-weight: bold; font-size: 18px;">Question
10:</span> <strong>What is the expected length? Does the length of the
majority ASVs fall within the expected range? Which range of sequences
length did you keep?</strong></p>
<p><span
style="color: green; font-weight: bold; font-size: 18px;">answer:</span>
The expected length is around 460 bps for the V3-V4 region based on the
primers used.</p>
<p>A significant number of ASVs fall within the range close to the
expected length. The highest counts appear to be in sequences that are
427 to 428 bps long, which is slightly shorter than the expected 460
bps. This discrepancy could be due to the trimming of primer sequences
or quality filtering that may have removed bases from the ends of
reads.</p>
<p>Sequences ranging from 300 to 460 bps, which should encompass the
majority of correctly sequenced V3-V4 region amplicons were kept.</p>
<p><span
style="color: red; font-weight: bold; font-size: 18px;">Question
11:</span> <strong>How many sequences were lost? And how abundant where
they?</strong></p>
<p><span
style="color: green; font-weight: bold; font-size: 18px;">answer:</span>
According to my results:</p>
<ul>
<li><strong>Number of sequences lost to chimera removal:</strong> 900
sequences.</li>
<li><strong>Total abundance of lost sequences:</strong> 29,494.99.</li>
</ul>
<p>This means that while you lost 900 sequences as chimeras, these
sequences collectively represented 29,494.99 in abundance before their
removal. This information is useful for understanding the impact of
chimera removal on your dataset and can be used to assess the quality
and reliability of the remaining sequences.</p>
<p><span
style="color: red; font-weight: bold; font-size: 18px;">Question
12:</span> <strong>Summary table. Where did the major loss of reads
happen?</strong></p>
<p><span
style="color: green; font-weight: bold; font-size: 18px;">answer:</span>
Based on the content of <code>summary_tab_pseudo.txt</code>, the major
loss of reads appears to have occurred for <code>mock9</code>, where
only 6.2% of the final percentage of reads were retained after the
entire DADA2 pipeline process. This is a significant drop compared to
the other samples, which retained between 42.1% and 87.8% of reads.</p>
<p>Here’s a breakdown of the read retention for <code>mock9</code>: -
<strong>DADA2 input:</strong> 83,980 reads - <strong>Filtered:</strong>
83,948 reads (almost no loss here, indicating filtering was not the
issue) - <strong>Denoised Forward and Reverse reads:</strong> 83,562 and
83,597 reads, respectively (again, very little loss, denoising seems
fine) - <strong>Merged:</strong> Only 5,207 reads (a substantial loss
occurs at this stage) - <strong>Non-chimeric (after chimera
removal):</strong> 5,203 reads (minimal loss here, indicating chimera
removal is not the issue) - <strong>Final percentage of reads
retained:</strong> 6.2%</p>
<p>The substantial loss at the merging stage suggests that there may be
an issue with the overlap of the reads, the quality of the reads at the
merging point, or other factors affecting the merging process. It would
be advisable to revisit the merging parameters and the quality of the
reads to understand why such a high loss occurred for
<code>mock9</code>.</p>
<div style="clear: both;">

</div>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
